
<!DOCTYPE html>

<html>
<head><style>.vega-actions a {margin-right: 12px;color: #757575;font-weight: normal;font-size: 13px;}.error {color: red;}</style><script src="https://cdn.jsdelivr.net/npm//vega@4" type="text/javascript"></script><script src="https://cdn.jsdelivr.net/npm//vega-lite@2.6.0" type="text/javascript"></script><script src="https://cdn.jsdelivr.net/npm//vega-embed@3" type="text/javascript"></script><meta content="text/html; charset=utf-8" http-equiv="Content-Type"/><meta content="IE=edge" http-equiv="X-UA-Compatible"/><meta content="width=device-width, initial-scale=1" name="viewport"/><link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/Font-Awesome-4.7/css/font-awesome.min.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/css/bootstrap.min.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/css/bootstrap-theme.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/images/gnomic_icon.png" rel="icon" type="image/png"/><script src="https://phenviz.navigome.com/js/jquery.min.js"></script><script src="https://phenviz.navigome.com/js/bootstrap.js"></script><title>Gene profile: ENSG00000146109</title></head>
<body>
<h4>Gene profile across phenotypes - ENSG00000146109 (ABT1)</h4><div id="vis"></div><div class="col-lg-12 col-md-12 content-item content-item-1"><p>This is a visualization of GWAS-derived gene associations  across Navigome phenotypes. Only phenotypes for which the gene has at least one significant association (p-value â‰¤ 0.05/20000) in <a href="https://ctg.cncr.nl/software/magma">MAGMA</a> <b>and/or</b> <a href="https://github.com/hakyimlab/MetaXcan">S-PrediXcan</a> analyses are shown. The first panel is a visualization of gene-phenotype associations obtained using <a href="https://ctg.cncr.nl/software/magma">MAGMA</a> (-log10(p-value)). Hover over any bar to highlight the phenotype in the other panels. The other panels are parallel coordinates visualizations for tissue-wise associations (z-scores)computed using <a href="https://github.com/hakyimlab/MetaXcan">S-PrediXcan</a>  and eQTL expression data from GTEx and DGN. Significance thresholds are delineated with black lines. The parallel coordinates show which phenotypes have a significant association and in which tissues, based on gene expression data. Clicking on a specific phenotype will lead to the reference of the GWAS study.</p></div>
<script type="text/javascript">
    var spec = {"config": {"view": {"width": 400, "height": 300}}, "vconcat": [{"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "bar", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "href": {"type": "nominal", "field": "phenotype_link"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "quantitative", "axis": {"title": "MAGMA -log10(p)"}, "field": "value"}, "y": {"type": "nominal", "axis": {"title": ""}, "field": "phenotype_reference"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-4e93ee1a91df87c61b7d73e211f0f4d1"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "x": {"type": "quantitative", "field": "ThresholdVal"}}}], "title": "MAGMA gene-wise association"}], "center": true, "data": {"name": "data-ead4b799129685f6a72a96c86ca8d403"}}, {"vconcat": [{"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-23.3771191798, 23.3771191798]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-23.3771191798, 23.3771191798]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-23.3771191798, 23.3771191798]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-23.3771191798, 23.3771191798]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Nervous system", "width": 500}], "center": true, "data": {"name": "data-6f2e320e4e9a67ad9e48503657bba758"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-23.3771191798, 23.3771191798]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-23.3771191798, 23.3771191798]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-23.3771191798, 23.3771191798]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-23.3771191798, 23.3771191798]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Digestive System", "width": 500}], "center": true, "data": {"name": "data-8bed04abb1c72178918bae1b63581f7e"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-23.3771191798, 23.3771191798]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-23.3771191798, 23.3771191798]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-23.3771191798, 23.3771191798]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-23.3771191798, 23.3771191798]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Blood, Arteries and Heart", "width": 500}], "center": true, "data": {"name": "data-27b481a6c9152394210ba91ad771ba48"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-23.3771191798, 23.3771191798]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-23.3771191798, 23.3771191798]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-23.3771191798, 23.3771191798]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-23.3771191798, 23.3771191798]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Adipose Tissue, Muscle, Skin and Cells", "width": 500}], "center": true, "data": {"name": "data-9889e4b779f11f8beefb43a2e66f7518"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-23.3771191798, 23.3771191798]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-23.3771191798, 23.3771191798]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-23.3771191798, 23.3771191798]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-23.3771191798, 23.3771191798]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Endocrine System and Others", "width": 500}], "center": true, "data": {"name": "data-b9e8ab8912cdae0b993f36a775ed9416"}}]}], "spacing": 40, "$schema": "https://vega.github.io/schema/vega-lite/v2.6.0.json", "datasets": {"data-ead4b799129685f6a72a96c86ca8d403": [{"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "magma_log10p", "value": 9.677408999999999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "magma_log10p", "value": 4.010929}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "magma_log10p", "value": 26.629006}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "magma_log10p", "value": 21.079105}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "magma_log10p", "value": 23.081597}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 9.340141000000001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 14.830001999999999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 12.4447}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 15.693168}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 25.072029}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 28.724090000000004}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 12.423613000000001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 20.739404}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 6.451193}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "magma_log10p", "value": 17.971876}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "magma_log10p", "value": 21.465454}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "magma_log10p", "value": 18.662701000000002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 47.241421}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "magma_log10p", "value": 71.048289}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "magma_log10p", "value": 61.090796999999995}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "magma_log10p", "value": 5.105201999999999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 13.26409}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 6.36227}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 43.032569}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "magma_log10p", "value": 31.610657}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "magma_log10p", "value": 5.177427}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "magma_log10p", "value": 4.2222040000000005}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 16.442985999999998}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "magma_log10p", "value": 15.835111999999999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "magma_log10p", "value": 9.606494999999999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 3.322129}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 2.842876}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 4.590371}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 2.393909}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 6.820649}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 3.19256}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 4.246830999999999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "magma_log10p", "value": 3.4225199999999996}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "magma_log10p", "value": 10.897806}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "magma_log10p", "value": 10.406825}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "magma_log10p", "value": 10.648997}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "magma_log10p", "value": 26.835765999999996}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "magma_log10p", "value": 51.696437}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 5.926465}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "magma_log10p", "value": 16.426954000000002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "magma_log10p", "value": 19.882861}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 4.716518}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "magma_log10p", "value": 2.390779}], "data-4e93ee1a91df87c61b7d73e211f0f4d1": [{"Threshold": "MAGMA -log10(p) \u2265 5.60206", "ThresholdVal": 5.60206}], "data-6f2e320e4e9a67ad9e48503657bba758": [{"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Amygdala", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Caudate", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Putamen", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cerebellum", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Cortex", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hippocampus", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Hypothalamus", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Nerve_Tibial", "value": 0.0}], "data-c4c9a654d45741ebc60aa567761bff19": [{"Text": "Significance Threshold", "Threshold": "PrediXcan zscore \u2265 4.70813", "ThresholdVal": 4.70813}], "data-4a3aa5bf8c5a41879455008930c25774": [{"Threshold": "PrediXcan zscore \u2264 -4.70813", "ThresholdVal": -4.70813}], "data-8bed04abb1c72178918bae1b63581f7e": [{"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Colon_Sigmoid", "value": 6.78825322538}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Colon_Sigmoid", "value": -7.26871906362}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Colon_Sigmoid", "value": 1.64400581657}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Colon_Sigmoid", "value": 0.272730556576}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Colon_Sigmoid", "value": -0.21548882075599998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": -2.2291869953599996}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": -1.08419414111}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": -2.25446807432}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": -2.0271278698299997}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": -1.4833843016600001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": -2.56379810544}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": -2.18093985271}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": -1.15689148339}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": -0.0584505850892}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Colon_Sigmoid", "value": -0.520714139274}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Colon_Sigmoid", "value": 1.1164167732799999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Colon_Sigmoid", "value": 0.0184143141615}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": 6.77886824539}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Colon_Sigmoid", "value": 13.4559842431}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Colon_Sigmoid", "value": 11.963464676300001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Colon_Sigmoid", "value": 2.9854091406}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": -2.2933231829400005}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": -0.15754494611}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": -6.724168987300001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Colon_Sigmoid", "value": 1.85455958945}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Colon_Sigmoid", "value": 4.368354867330001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Colon_Sigmoid", "value": -1.83733622716}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": -2.21513969092}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Colon_Sigmoid", "value": 8.36205647265}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Colon_Sigmoid", "value": 6.160933751519999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": 4.35345463574}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": 3.9643926070599997}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": 5.22992852191}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": 3.66432822263}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": 5.894870973119999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": 4.866548363430001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": 5.230203455990001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Colon_Sigmoid", "value": 0.815081537635}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Colon_Sigmoid", "value": -3.49659424925}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Colon_Sigmoid", "value": 1.16937387327}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Colon_Sigmoid", "value": 7.722578256769999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Colon_Sigmoid", "value": -0.061263274901300006}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Colon_Sigmoid", "value": 1.24481417309}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": -1.7755083172200001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Colon_Sigmoid", "value": -1.04130061988}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Colon_Sigmoid", "value": -1.52195793907}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": -1.84583718928}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Sigmoid", "value": 4.0471098539599994}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Colon_Transverse", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Esophagus_Mucosa", "value": 4.5180180211500005}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Esophagus_Mucosa", "value": -1.6341622745}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Esophagus_Mucosa", "value": 2.77110700356}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Esophagus_Mucosa", "value": -0.10018893189200001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Esophagus_Mucosa", "value": -0.724833589486}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": -1.0704243002399998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": -2.8614336577599997}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": -1.1695819313299998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": -1.05211776435}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": 0.799218773682}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": -0.906351161934}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": -1.6649653592299998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": -3.24767454131}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": -2.33622511747}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Esophagus_Mucosa", "value": 1.9875674552200002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Esophagus_Mucosa", "value": 0.7057751486020001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Esophagus_Mucosa", "value": 0.260874380134}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": 4.97514642284}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Esophagus_Mucosa", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Esophagus_Mucosa", "value": 8.12220515457}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Esophagus_Mucosa", "value": 2.78422019393}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": -1.9336263852900002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": 1.9245522704499998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": -2.60114807262}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Esophagus_Mucosa", "value": 1.05376451672}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Esophagus_Mucosa", "value": 3.3943689768799996}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Esophagus_Mucosa", "value": -3.7250233433199997}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": -2.46233112559}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Esophagus_Mucosa", "value": 5.81208005674}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Esophagus_Mucosa", "value": 5.0181169509300005}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": 3.32095385614}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": 3.1840852283200003}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": 3.13077877638}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": 4.599566231130001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": 5.04582104866}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": 3.05644731563}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": 4.86762155115}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Esophagus_Mucosa", "value": 0.870677329847}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Esophagus_Mucosa", "value": -3.1853006381}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Esophagus_Mucosa", "value": 2.1486424349799997}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Esophagus_Mucosa", "value": 5.75786467209}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Esophagus_Mucosa", "value": -0.161004008696}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Esophagus_Mucosa", "value": 0.176721628226}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": -2.12451540779}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Esophagus_Mucosa", "value": -1.5345967862399998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Esophagus_Mucosa", "value": -2.07055571083}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": -0.594011460599}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Mucosa", "value": 2.1734911822400003}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Esophagus_Muscularis", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Liver", "value": 3.1691954044300004}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Liver", "value": -2.7597602065099998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Liver", "value": 5.64630601475}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Liver", "value": 4.1317982009500005}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Liver", "value": -0.7498682595720001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": -2.31168690879}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": -4.5655352615}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": -2.75248121949}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": -2.62443558594}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": -1.06249975555}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": -2.84327590848}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": -1.99711045553}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": -4.12890397654}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": -2.6004770893}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Liver", "value": 3.84811773984}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Liver", "value": 4.79204190364}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Liver", "value": 4.30963746346}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": 7.26720521385}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Liver", "value": 12.306643485999999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Liver", "value": 10.6560718694}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Liver", "value": 5.6934183246000005}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": -2.31049353634}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": 2.57330192132}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": -5.022168311140001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Liver", "value": 5.30959872035}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Liver", "value": 1.17166428941}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Liver", "value": -0.9046556209409999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": -3.6232524231}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Liver", "value": 3.37756997441}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Liver", "value": 4.25260621965}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": 2.82452839182}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": 2.8272959959799997}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": 4.19785992752}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": 4.4266933581800005}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": 5.457755097840001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": 3.8903738843599998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": 5.15733037852}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Liver", "value": 2.81526127834}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Liver", "value": -6.44784101011}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Liver", "value": 4.4984336658199995}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Liver", "value": 2.97330012009}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Liver", "value": -1.69601173963}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Liver", "value": -1.1208219962100001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": -1.09876674425}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Liver", "value": -0.42609015342900003}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Liver", "value": -0.7020666591820001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": -2.86646058265}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Liver", "value": 2.77900147273}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Minor_Salivary_Gland", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Stomach", "value": 0.0}], "data-27b481a6c9152394210ba91ad771ba48": [{"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "DGN_Whole_Blood", "value": 6.1394112103316525}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "DGN_Whole_Blood", "value": -3.5641626145899608}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "DGN_Whole_Blood", "value": 2.121333591590817}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "DGN_Whole_Blood", "value": 0.343791021043564}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "DGN_Whole_Blood", "value": -2.9977430414485964}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": -2.2702202815149817}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": -4.336700326910668}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": -2.4961944234046727}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": -2.5210899994986766}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": -1.318183358343335}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": -3.0071654592935424}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": -2.3365785559327965}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": -4.273684949304491}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": -3.0140051136301365}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "DGN_Whole_Blood", "value": 0.5832678531102767}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "DGN_Whole_Blood", "value": 1.4288750568373014}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "DGN_Whole_Blood", "value": 0.851875242723016}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": 4.123055727581667}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "DGN_Whole_Blood", "value": 8.51214137606634}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "DGN_Whole_Blood", "value": 7.653358635346347}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "DGN_Whole_Blood", "value": 1.7296659175686684}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": -2.6135126475299892}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": 2.676562643133434}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": -2.058203973066596}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "DGN_Whole_Blood", "value": 0.97243310772546}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "DGN_Whole_Blood", "value": 5.163999672016886}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "DGN_Whole_Blood", "value": -1.7163036931598925}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": -1.9660456700764064}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "DGN_Whole_Blood", "value": 7.390081998514029}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "DGN_Whole_Blood", "value": 1.4585638220052377}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": 2.707888668114492}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": 2.6726471066559987}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": 1.55405763387824}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": 2.886551799934808}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": 3.1363806376841312}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": 1.5721973431981031}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": 2.9338231580129657}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "DGN_Whole_Blood", "value": 0.9542918938113362}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "DGN_Whole_Blood", "value": -3.8164336669170105}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "DGN_Whole_Blood", "value": 1.2119203551656454}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "DGN_Whole_Blood", "value": 5.9690266095331435}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "DGN_Whole_Blood", "value": 1.0252603071017714}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "DGN_Whole_Blood", "value": 1.7217082076723906}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": -2.074006930783804}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "DGN_Whole_Blood", "value": -3.118781085052241}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "DGN_Whole_Blood", "value": -3.552260603969428}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": 1.2290920497611346}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "DGN_Whole_Blood", "value": 1.0611760372943746}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Whole_Blood", "value": 2.62696560784}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Whole_Blood", "value": 0.201679732419}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Whole_Blood", "value": 3.1634663718700002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Whole_Blood", "value": 0.156478963432}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Whole_Blood", "value": -1.7267952143599998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": 0.467153801146}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": 0.136576446808}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": 0.5900079407609999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": 0.44464051340599997}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": 1.3251620958399999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": 0.882853394694}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": -1.90677482415}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": -2.61767341365}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": -1.5598006804799998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Whole_Blood", "value": 2.64276075257}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Whole_Blood", "value": 1.8453336513}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Whole_Blood", "value": 1.52822356979}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": 4.62389833477}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Whole_Blood", "value": 8.46057537991}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Whole_Blood", "value": 7.50213129178}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Whole_Blood", "value": -0.013742424215899999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": -2.0175682338099996}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": 1.47416737867}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": -5.269093918299999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Whole_Blood", "value": 1.90833353553}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Whole_Blood", "value": 2.66891559728}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Whole_Blood", "value": -2.18361333669}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": 1.05335154373}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Whole_Blood", "value": 3.00122069848}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Whole_Blood", "value": -0.292574402337}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": 0.47218408011600005}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": 0.435814975602}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": -0.378386629414}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": 1.51224250428}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": 1.19577219822}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": -0.372497177844}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": 1.11145785197}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Whole_Blood", "value": 0.0861035336309}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Whole_Blood", "value": -4.59983980137}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Whole_Blood", "value": 2.6817376355700002}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Whole_Blood", "value": 2.41638140213}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Whole_Blood", "value": -0.164792909382}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Whole_Blood", "value": -1.2087256436799998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": -1.61846817912}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Whole_Blood", "value": -2.2287734877200003}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Whole_Blood", "value": -2.09788044493}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": 1.6471854576700002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Whole_Blood", "value": -0.740948203443}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Aorta", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Coronary", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Artery_Tibial", "value": 3.05052008894}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Artery_Tibial", "value": -2.48431881157}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Artery_Tibial", "value": 2.76184301818}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Artery_Tibial", "value": -0.218064449101}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Artery_Tibial", "value": -0.119641707788}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": -0.49750685529700006}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": -1.9601187138599998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": -0.532329528169}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": -0.555469959336}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": 1.15964713093}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": -0.436018615476}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": -0.930166164474}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": -2.63184230032}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": -1.91899883241}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Artery_Tibial", "value": 1.6463401970700002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Artery_Tibial", "value": 0.491917592236}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Artery_Tibial", "value": -0.08371470100660001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": 5.64783122079}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Artery_Tibial", "value": 9.157097632560001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Artery_Tibial", "value": 7.99000481737}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Artery_Tibial", "value": 1.24314632159}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": -1.2337543685799999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": 1.69043083496}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": -4.07300830056}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Artery_Tibial", "value": 1.3612264520500001}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Artery_Tibial", "value": 3.02778279098}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Artery_Tibial", "value": 0.751850554738}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": -0.905866138803}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Artery_Tibial", "value": 3.49054015498}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Artery_Tibial", "value": -1.1294778887299999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": 1.3924697962700001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": 1.4548971573299998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": 1.34683835838}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": -0.848853976631}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": 0.542750864976}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": 1.44727235576}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": 0.571940131869}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Artery_Tibial", "value": 1.06183132177}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Artery_Tibial", "value": -0.669612250714}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Artery_Tibial", "value": -1.31438870006}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Artery_Tibial", "value": 2.60415516308}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Artery_Tibial", "value": -1.04015734869}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Artery_Tibial", "value": -1.25171227335}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": -1.6574713545500002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Artery_Tibial", "value": 1.4146443279499998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Artery_Tibial", "value": 0.94974717923}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": 2.5616533745}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Artery_Tibial", "value": 0.0256272522363}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Spleen", "value": 0.0}], "data-9889e4b779f11f8beefb43a2e66f7518": [{"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Adipose_Subcutaneous", "value": 4.48015869486}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Adipose_Subcutaneous", "value": -2.35140671061}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Adipose_Subcutaneous", "value": 1.52294139601}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Adipose_Subcutaneous", "value": -0.295252162282}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Adipose_Subcutaneous", "value": 0.0808140853786}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": 0.8590540065569999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": 0.210377957077}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": 1.0982819083}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": 1.18763327543}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": 2.9090814167000003}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": 1.83068286727}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": -0.559146199139}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": -1.2868739577600001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": -1.12696352655}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Adipose_Subcutaneous", "value": 1.5298504196}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Adipose_Subcutaneous", "value": 1.13575131723}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Adipose_Subcutaneous", "value": 0.649113788328}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": 3.24153019588}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Adipose_Subcutaneous", "value": 8.47429413682}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Adipose_Subcutaneous", "value": 8.15395979486}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Adipose_Subcutaneous", "value": 1.7229141479700003}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": -0.6436451321}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": 0.8141307202120001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": -3.12604320723}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Adipose_Subcutaneous", "value": 1.31122084224}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Adipose_Subcutaneous", "value": 3.98498721308}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Adipose_Subcutaneous", "value": -1.12631740113}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": -0.5352089091}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Adipose_Subcutaneous", "value": 5.76289260704}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Adipose_Subcutaneous", "value": 4.23831428009}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": 3.4980785550099998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": 3.4068480144200004}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": 3.0768778244300004}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": 4.46663863013}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": 5.07372634874}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": 3.13490666083}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": 4.97367378568}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Adipose_Subcutaneous", "value": 1.47942973847}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Adipose_Subcutaneous", "value": -3.86801425018}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Adipose_Subcutaneous", "value": 2.42866704562}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Adipose_Subcutaneous", "value": 5.8214727188}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Adipose_Subcutaneous", "value": -1.30851738522}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Adipose_Subcutaneous", "value": -1.42134237771}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": -2.03563448462}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Adipose_Subcutaneous", "value": -1.40700906587}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Adipose_Subcutaneous", "value": -1.70994165104}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": -0.38059089916199995}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Subcutaneous", "value": 2.09567394128}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Adipose_Visceral_Omentum", "value": 4.60764337498}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Adipose_Visceral_Omentum", "value": -1.70931240907}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Adipose_Visceral_Omentum", "value": 5.75400635259}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Adipose_Visceral_Omentum", "value": 3.14842329982}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Adipose_Visceral_Omentum", "value": -0.381179439696}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": -0.301498303864}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": -0.758831295326}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": -0.22317863385000003}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": -0.0855645892355}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": 2.12190656393}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": 0.434493474436}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": -0.312272489955}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": -0.580114075396}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": -0.570546811368}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Adipose_Visceral_Omentum", "value": 3.5596051484699993}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Adipose_Visceral_Omentum", "value": 2.70107727198}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Adipose_Visceral_Omentum", "value": 2.39589483873}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": 6.980542968080001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Adipose_Visceral_Omentum", "value": 11.1241858365}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Adipose_Visceral_Omentum", "value": 9.16795273956}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Adipose_Visceral_Omentum", "value": 0.836290105342}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": -0.38575192202299996}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": 0.169194683744}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": -6.38573434829}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Adipose_Visceral_Omentum", "value": 3.31589276028}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Adipose_Visceral_Omentum", "value": 3.40349837468}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Adipose_Visceral_Omentum", "value": -0.558960418866}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": -1.7553375407799998}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Adipose_Visceral_Omentum", "value": 5.22834904526}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Adipose_Visceral_Omentum", "value": 3.4057727351}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": 2.60440314203}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": 2.7312702940299998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": 3.20655076023}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": 2.9586867903200003}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": 4.339632158630001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": 3.3072424059900003}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": 4.24321972329}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Adipose_Visceral_Omentum", "value": 1.7559331751700002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Adipose_Visceral_Omentum", "value": -5.72795785553}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Adipose_Visceral_Omentum", "value": 4.82380887636}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Adipose_Visceral_Omentum", "value": 4.303827463269999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Adipose_Visceral_Omentum", "value": -1.26556827059}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Adipose_Visceral_Omentum", "value": -2.50070213251}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": -2.81461632207}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Adipose_Visceral_Omentum", "value": -0.144490639004}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Adipose_Visceral_Omentum", "value": -0.350278849868}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": -0.983887412993}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adipose_Visceral_Omentum", "value": 2.2696419804400003}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Breast_Mammary_Tissue", "value": 3.7790797242800003}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Breast_Mammary_Tissue", "value": -1.42295147572}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Breast_Mammary_Tissue", "value": -7.51304273632}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Breast_Mammary_Tissue", "value": -3.8223091898800003}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Breast_Mammary_Tissue", "value": -2.47552527274}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": -2.1091469573299997}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": -1.4308830589300001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": -2.1057311010900004}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": -2.16107953428}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": 1.2374939459}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": -1.59848564867}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": -2.1554947124}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": -1.66149561757}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": -0.611186602164}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Breast_Mammary_Tissue", "value": -5.0482460510900005}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Breast_Mammary_Tissue", "value": -1.2486446202700001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Breast_Mammary_Tissue", "value": -1.51047543693}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": -6.55438048389}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Breast_Mammary_Tissue", "value": -7.11509163375}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Breast_Mammary_Tissue", "value": -4.67097872353}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Breast_Mammary_Tissue", "value": 0.0251036329844}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": -2.32841060372}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": 0.314183469412}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": 5.47717236706}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Breast_Mammary_Tissue", "value": -3.4183315156700003}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Breast_Mammary_Tissue", "value": 3.8337556497199996}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Breast_Mammary_Tissue", "value": -1.31762492675}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": -0.563761124722}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Breast_Mammary_Tissue", "value": 5.25229218995}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Breast_Mammary_Tissue", "value": 1.79104427225}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": 2.69291354941}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": 2.4499565535}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": 1.95587825491}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": 1.16787401451}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": 2.23114511005}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": 1.79746484175}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": 1.9689123489900002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Breast_Mammary_Tissue", "value": -2.701347254}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Breast_Mammary_Tissue", "value": 7.490366390689998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Breast_Mammary_Tissue", "value": -10.694376871000001}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Breast_Mammary_Tissue", "value": 5.40711824842}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Breast_Mammary_Tissue", "value": -0.727499551861}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Breast_Mammary_Tissue", "value": 0.728426181848}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": -1.07753383594}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Breast_Mammary_Tissue", "value": -0.839912000829}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Breast_Mammary_Tissue", "value": -1.2476293633600002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": -5.18443669765}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Breast_Mammary_Tissue", "value": 1.24688167536}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Muscle_Skeletal", "value": 6.85879610844}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Muscle_Skeletal", "value": -4.4996624688599995}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Muscle_Skeletal", "value": 0.366615391444}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Muscle_Skeletal", "value": -0.7478266582719999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Muscle_Skeletal", "value": -1.1468218127299998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": -2.65716710837}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": -2.38190478218}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": -2.6923792071499997}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": -2.50073999035}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": -1.6479419062399998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": -3.2140523113599997}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": -2.7095631278400005}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": -2.63499914342}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": -1.03903549161}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Muscle_Skeletal", "value": -0.710794346278}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Muscle_Skeletal", "value": 0.275717226379}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Muscle_Skeletal", "value": -0.449228925404}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": 3.4411380642300005}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Muscle_Skeletal", "value": 7.73023024495}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Muscle_Skeletal", "value": 7.08608534623}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Muscle_Skeletal", "value": 3.6958348286900002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": -2.7909597975200002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": 0.836702922436}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": -0.9258148742450001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Muscle_Skeletal", "value": 0.347365174806}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Muscle_Skeletal", "value": 4.44708520682}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Muscle_Skeletal", "value": -4.13475911119}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": -4.5554760691}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Muscle_Skeletal", "value": 8.20950978856}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Muscle_Skeletal", "value": 7.18346728728}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": 4.255987257619999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": 4.18300954139}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": 4.26191775164}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": 4.32437710845}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": 5.8012188814600005}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": 4.23714183856}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": 5.591069156490001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Muscle_Skeletal", "value": 0.110157941733}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Muscle_Skeletal", "value": -1.43046091614}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Muscle_Skeletal", "value": 0.5049472885}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Muscle_Skeletal", "value": 7.35914075173}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Muscle_Skeletal", "value": 0.696172834481}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Muscle_Skeletal", "value": -0.0903672353202}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": -3.4725228090900004}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Muscle_Skeletal", "value": -2.81980244238}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Muscle_Skeletal", "value": -3.2119559051700004}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": -1.2133669827299998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Muscle_Skeletal", "value": 2.76133109639}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Cells_Transformed_fibroblasts", "value": 5.9684833174500005}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Cells_Transformed_fibroblasts", "value": -3.1534835984599994}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Cells_Transformed_fibroblasts", "value": 3.8286330635100003}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Cells_Transformed_fibroblasts", "value": 1.48985080038}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Cells_Transformed_fibroblasts", "value": -0.988901446301}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": -1.2277574275}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": -2.16802226776}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": -1.25608844772}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": -1.12889118748}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": -0.71258280893}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": -1.56496678891}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": -1.1188253757200002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": -2.17997910468}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": -1.41330134046}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Cells_Transformed_fibroblasts", "value": 1.89095265498}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Cells_Transformed_fibroblasts", "value": 1.21219483435}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Cells_Transformed_fibroblasts", "value": 0.725717627633}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": 6.018577260419999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Cells_Transformed_fibroblasts", "value": 10.2492285003}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Cells_Transformed_fibroblasts", "value": 8.62228077413}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Cells_Transformed_fibroblasts", "value": 2.41986058082}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": -1.3084271909799998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": 1.15680224041}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": -3.52383615067}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Cells_Transformed_fibroblasts", "value": 2.13459933714}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Cells_Transformed_fibroblasts", "value": 4.02535473194}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Cells_Transformed_fibroblasts", "value": -5.00947095573}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": -2.75382676532}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Cells_Transformed_fibroblasts", "value": 7.38610633332}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Cells_Transformed_fibroblasts", "value": 5.440736719969999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": 3.54297121943}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": 3.44899326683}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": 4.57831835544}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": 4.24955431738}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": 6.08870441672}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": 4.39335611753}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": 5.811501215140001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Cells_Transformed_fibroblasts", "value": 1.3436840791799998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Cells_Transformed_fibroblasts", "value": -2.93861420121}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Cells_Transformed_fibroblasts", "value": 3.0864722255199997}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Cells_Transformed_fibroblasts", "value": 6.588640139269999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Cells_Transformed_fibroblasts", "value": -0.465963367753}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Cells_Transformed_fibroblasts", "value": 1.1990591124799999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": -1.60389479051}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Cells_Transformed_fibroblasts", "value": -1.55715563332}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Cells_Transformed_fibroblasts", "value": -1.9855443363799998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": -1.29866061545}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_Transformed_fibroblasts", "value": 3.3368696577599994}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 5.68669295885}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -5.94837966166}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 9.80684839047}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 5.72531219222}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.908133094672}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.714586324033}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -0.954749369976}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.694233004877}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.87685526271}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -1.78972279939}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -0.2540171031}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -0.548243226102}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -2.4748823226299996}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -1.9318504540099999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 5.15117154751}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 4.5536829484300005}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 3.54276049301}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 13.459740628599999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 23.3771191798}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 19.7273663119}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 1.0853775265899999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -0.568173558828}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 1.86261765375}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -13.1903979926}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 7.496404533330001}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 2.82550698953}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -2.56700066699}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -1.51614519085}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 6.509804485830001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 5.147641195}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 3.65334652188}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 3.6286815372400003}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 4.34229203994}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 4.751322482769999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 6.03005437952}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 4.3440049228}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 5.8477825902}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 2.1825683623599996}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -7.403091021}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 8.05305618175}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 5.1497222031800005}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.15333732052000001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 4.071223637109999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -2.83704062196}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -4.3673003085}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": -4.85670545409}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 4.9735465488}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 4.10210883666}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 6.6496590992}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -3.90072707426}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.2178115392400004}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -0.135308441842}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -4.25141212606}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -2.74725126787}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -5.04647447378}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -3.0875339330799996}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -3.37755164308}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -4.06596600613}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -4.6230366828300005}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -2.1500937808900003}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -4.13965636937}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -2.76234843297}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.3283551599799999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.18326102303300001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -0.44617401835299997}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 5.96301925485}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 10.925349748599999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 9.534354595910001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.5797274179599996}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -2.4620675898099997}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.53497248632}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -4.1641916534800005}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.951528059648}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 5.0882874966000005}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -2.4235822862000003}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -1.99853505035}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 8.243118414380001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.5863144880899998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.65541242676}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.4469640359900002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.2611217839700002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.5913807136899998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.00264669963}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.16490353092}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.8531731654599997}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -0.703893936208}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.195381256137}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0466815795144}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 7.29787411617}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.859275436458}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.0224910751599996}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -1.99156863035}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -2.61342462247}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -3.09775238593}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.18939565362}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 1.11445811653}], "data-b9e8ab8912cdae0b993f36a775ed9416": [{"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Adrenal_Gland", "value": 5.477630974609999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Adrenal_Gland", "value": -3.15284807496}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Adrenal_Gland", "value": 4.62330255565}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Adrenal_Gland", "value": 2.90159071419}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Adrenal_Gland", "value": 0.506667608749}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": -1.2898993436}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": -1.81160933843}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": -1.22300479438}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": -0.915375442879}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": -0.080133071788}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": -1.20931051795}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": -1.0972116074}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": -1.88012989102}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": -1.04870514694}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Adrenal_Gland", "value": 2.2096571082799996}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Adrenal_Gland", "value": 3.32691571399}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Adrenal_Gland", "value": 2.79093773009}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": 7.780280381569999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Adrenal_Gland", "value": 11.265235246500001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Adrenal_Gland", "value": 8.9284814736}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Adrenal_Gland", "value": 3.11683209161}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": -1.23904112223}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": 0.864998055733}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": -4.08854435452}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Adrenal_Gland", "value": 4.07607984515}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Adrenal_Gland", "value": 3.23489249866}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Adrenal_Gland", "value": -3.0611208617}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": -4.09205943761}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Adrenal_Gland", "value": 5.80283234008}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Adrenal_Gland", "value": 3.1915710169400002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": 3.12054587467}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": 2.97115234943}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": 2.56840003708}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": 2.80930751675}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": 3.6625871348400003}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": 2.41350187931}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": 3.28445894314}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Adrenal_Gland", "value": 3.01094978299}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Adrenal_Gland", "value": -6.012981470830001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Adrenal_Gland", "value": 3.8728340323800006}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Adrenal_Gland", "value": 3.8434648064099997}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Adrenal_Gland", "value": -0.9625696482739999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Adrenal_Gland", "value": -1.83879797634}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": -4.40109147257}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Adrenal_Gland", "value": 0.5054971126810001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Adrenal_Gland", "value": 0.36691661534000003}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": 0.574760715128}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Adrenal_Gland", "value": 2.52131378651}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Pituitary", "value": 6.02412896102}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Pituitary", "value": -3.1551832591299998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Pituitary", "value": 4.07370233136}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Pituitary", "value": 1.3570392671399998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Pituitary", "value": -0.7072472797689999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": -0.896696096702}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": -2.18592782399}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": -0.948818186226}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": -0.792913559639}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": -0.864777863366}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": -1.3780820588299998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": -0.814748705687}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": -1.98777999708}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": -1.64694480329}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Pituitary", "value": 2.0290725330900004}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Pituitary", "value": 1.00944609917}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Pituitary", "value": 0.535743017049}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": 5.9247769428999995}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Pituitary", "value": 10.4652855807}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Pituitary", "value": 8.9197015635}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Pituitary", "value": 3.85910601416}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": -0.9938230704559999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": 1.2458421555399999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": -3.19413004634}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Pituitary", "value": 1.9512558521}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Pituitary", "value": 4.28942568356}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Pituitary", "value": -4.26380819436}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": -3.72196035388}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Pituitary", "value": 7.6065776631199995}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Pituitary", "value": 7.08899605844}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": 4.19402829094}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": 4.06591316102}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": 4.62194534895}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": 4.40274164459}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": 6.17003342663}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": 4.57964919919}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": 6.01616078651}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Pituitary", "value": 1.38779555589}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Pituitary", "value": -3.36871520875}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Pituitary", "value": 3.1544272500800004}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Pituitary", "value": 7.2079707914}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Pituitary", "value": -0.138599022046}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Pituitary", "value": 0.245648829971}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": -2.92705621588}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Pituitary", "value": -1.34248590748}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Pituitary", "value": -1.8190064134200001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": -0.9875301876260001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pituitary", "value": 3.6439052391300004}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Pancreas", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Thyroid", "value": 5.1075071886800005}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Thyroid", "value": -1.51995916813}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Thyroid", "value": 6.647824755909999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Thyroid", "value": 5.13478021121}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Thyroid", "value": 1.8784753917900001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": 0.0226056467452}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": -1.28791937884}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": 0.0331103505608}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": 0.325200595483}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": 2.99637459655}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": 1.16494627662}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": -0.1713507174}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": -1.37437850122}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": -1.1960186427700001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Thyroid", "value": 4.25463355866}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Thyroid", "value": 4.61960806166}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Thyroid", "value": 4.34829387058}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": 6.71319964381}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Thyroid", "value": 11.468425463099999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Thyroid", "value": 9.872076568110002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Thyroid", "value": 2.59623904117}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": -0.405484632259}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": 1.02738631267}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": -5.64410245691}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Thyroid", "value": 5.61994006637}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Thyroid", "value": 3.39116209345}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Thyroid", "value": -1.12579029256}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": -3.6846495759900004}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Thyroid", "value": 5.9898426351600005}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Thyroid", "value": 5.5564662055400005}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": 4.58923568083}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": 4.33945207519}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": 3.39667369852}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": 4.443870682169999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": 5.39141655441}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": 3.1817315018700003}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": 5.01519099444}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Thyroid", "value": 1.96516183435}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Thyroid", "value": -4.6899729617000006}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Thyroid", "value": 2.7781056656}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Thyroid", "value": 5.74192921888}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Thyroid", "value": -3.08742754976}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Thyroid", "value": -3.17362154307}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": -1.27844408506}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Thyroid", "value": 0.337024555123}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Thyroid", "value": -0.08944283985780001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": -1.81908851022}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Thyroid", "value": 4.2177508073}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Lung", "value": 6.08637336424}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Lung", "value": -2.90653926357}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Lung", "value": 4.77813896349}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Lung", "value": 1.3945685238299999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Lung", "value": -0.561327634681}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": -1.2037060959399999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": -2.52755525699}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": -1.25566626366}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": -1.1114922036200001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": 0.574070529495}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": -1.09073823147}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": -1.62390163412}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": -3.06065254552}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": -1.9497293242400002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Lung", "value": 2.68872647018}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Lung", "value": 1.7055072661500001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Lung", "value": 1.21764808203}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": 6.18397599982}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Lung", "value": 11.355296333399998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Lung", "value": 9.87536376643}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Lung", "value": 2.74430298074}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": -1.83859609978}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": 1.7973076446599998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": -4.486529001519999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Lung", "value": 2.29409609635}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Lung", "value": 4.22192278815}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Lung", "value": -3.5675212802900003}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": -2.95279670922}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Lung", "value": 7.46853344641}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Lung", "value": 4.696435053359999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": 3.6223269724500002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": 3.43290726464}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": 4.19294486874}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": 4.14319642899}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": 5.584763658969999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": 3.66198510732}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": 5.13410655747}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Lung", "value": 1.77392720093}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Lung", "value": -4.41736328832}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Lung", "value": 4.16874839655}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Lung", "value": 7.010228461950001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Lung", "value": -0.783464318139}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Lung", "value": -0.590880766891}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": -1.6251702925}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Lung", "value": -1.59566217162}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Lung", "value": -2.14553716531}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": -2.23278254306}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Lung", "value": 3.3265318536799997}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Ovary", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Prostate", "value": 0.9305563304819999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Prostate", "value": -0.47101738303999996}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Prostate", "value": 11.0146953141}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Prostate", "value": 7.361411488260001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Prostate", "value": 3.7504895951699995}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": 1.4360851548299998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": -0.589397282331}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": 1.51702798637}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": 1.9376674656}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": 4.47855581563}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": 3.13346639218}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": 1.57519512037}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": -0.451909535624}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": -0.999762189403}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Prostate", "value": 7.626513038660001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Prostate", "value": 6.024328869990001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Prostate", "value": 5.89436849561}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": 9.96481575218}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Prostate", "value": 15.143927081}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Prostate", "value": 12.6115160776}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Prostate", "value": 3.1650825724500002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": 1.41056846276}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": 1.07180633576}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": -8.43098525721}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Prostate", "value": 7.6608578183}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Prostate", "value": -0.503873825269}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Prostate", "value": 0.9519254616949999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": -1.28556837529}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Prostate", "value": 1.0560591402}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Prostate", "value": 3.36038615064}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": 2.8670383033599998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": 2.8031133438099998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": 2.83676023082}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": 3.5280146302800004}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": 4.31502440502}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": 2.76151525116}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": 4.194170135909999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Prostate", "value": 5.03423411102}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Prostate", "value": -8.53938224206}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Prostate", "value": 6.945749061760001}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Prostate", "value": 1.71275040042}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Prostate", "value": -3.41321841161}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Prostate", "value": -6.33676475372}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": -2.9570479752999996}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Prostate", "value": 4.165767581569999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Prostate", "value": 4.00143048539}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": -0.9584224776710001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Prostate", "value": 2.09368815742}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Testis", "value": 6.53163480051}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Testis", "value": -1.13293321441}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Testis", "value": 3.3184875599400003}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Testis", "value": 4.13334513571}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Testis", "value": 1.42622667722}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": -0.231606610556}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": -0.90760185416}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": -0.265890844202}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": 0.11933668019000002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": 2.3037721156599997}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": 0.679946536464}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": -0.393480932228}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": -1.3842836792200002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": -1.1168537933299998}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Testis", "value": 1.8975342813900002}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Testis", "value": 2.8771339033200003}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Testis", "value": 2.5833179556}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": 5.4736595456599995}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Testis", "value": 8.28123979193}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Testis", "value": 6.723282086499999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Testis", "value": 2.6460953304599997}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": -0.566038098094}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": 0.863849133398}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": -2.68286194118}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Testis", "value": 3.69130106873}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Testis", "value": 3.91448230927}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Testis", "value": -2.20756102547}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": -5.420040067}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Testis", "value": 7.84530670634}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Testis", "value": 8.41842267954}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": 5.05966976461}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": 4.83011139894}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": 5.936229045759999}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": 3.80202500047}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": 6.75689321775}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": 5.71174973752}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": 6.4597053725}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Testis", "value": 2.52504355569}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Testis", "value": -2.46695788961}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Testis", "value": 2.25999531949}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Testis", "value": 7.29493338631}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Testis", "value": -1.28648476956}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Testis", "value": -2.99366508767}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": -2.3254410291900003}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Testis", "value": 1.19026134498}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Testis", "value": 0.6717817398370001}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": -3.10296005072}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Testis", "value": 4.796739888389999}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Uterus", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 0, "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Intercept": "1.0594 (0.0166)", "LDSC Lambda GC": 2.0217, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "code": "BFPC04", "category": "b: anthropometric", "phenotype": "Body fat percentage complete", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5273, "association": 9.677408999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0389 (0.0109)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 1, "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Intercept": "1.0021 (0.0255)", "LDSC Lambda GC": 1.2464, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "code": "BLOO02", "category": "g: blood", "phenotype": "red blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6473, "association": 4.010929, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0032 (0.0394)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 2, "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "LDSC Intercept": "1.0092 (0.0233)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "code": "BLOO03", "category": "g: blood", "phenotype": "hemoglobin concentration", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.4634, "association": 26.629006, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0199 (0.0502)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 3, "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Intercept": "1.0155 (0.0407)", "LDSC Lambda GC": 1.207, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "code": "BLOO04", "category": "g: blood", "phenotype": "reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6377, "association": 21.079105, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0243 (0.0638)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 4, "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Intercept": "1.0465 (0.0338)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "code": "BLOO05", "category": "g: blood", "phenotype": "monocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6776, "association": 23.081597, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0686 (0.0499)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 5, "phenotype_reference": "Neutrophil count (BLOO06) ", "LDSC Intercept": "0.9567 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1582 (0.0186)", "code": "BLOO06", "category": "g: blood", "phenotype": "neutrophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.516, "association": 9.340141000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 6, "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Intercept": "0.9587 (0.0273)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "code": "BLOO07", "category": "g: blood", "phenotype": "eosinophil count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6196, "association": 14.830001999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 7, "phenotype_reference": "Granulocyte count (BLOO09) ", "LDSC Intercept": "0.9591 (0.0206)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "code": "BLOO09", "category": "g: blood", "phenotype": "granulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5217, "association": 12.4447, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 8, "phenotype_reference": "Myeloid white cell count (BLOO10) ", "LDSC Intercept": "0.9623 (0.02)", "LDSC Lambda GC": 1.2168, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "code": "BLOO10", "category": "g: blood", "phenotype": "myeloid white cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5375, "association": 15.693168, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 9, "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Intercept": "0.9683 (0.0197)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "code": "BLOO11", "category": "g: blood", "phenotype": "lymphocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6107, "association": 25.072029, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 10, "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Intercept": "0.9648 (0.0192)", "LDSC Lambda GC": 1.2431, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "code": "BLOO12", "category": "g: blood", "phenotype": "white blood cell count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5935, "association": 28.724090000000004, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 11, "phenotype_reference": "Sum basophil neutrophil counts (BLOO13) ", "LDSC Intercept": "0.9585 (0.0207)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1592 (0.0186)", "code": "BLOO13", "category": "g: blood", "phenotype": "Sum basophil neutrophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5207, "association": 12.423613000000001, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 12, "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Intercept": "0.9694 (0.0258)", "LDSC Lambda GC": 1.1908, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "code": "BLOO16", "category": "g: blood", "phenotype": "Sum eosinophil basophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5912, "association": 20.739404, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 13, "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Intercept": "0.9588 (0.0243)", "LDSC Lambda GC": 1.1876, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "code": "BLOO18", "category": "g: blood", "phenotype": "Eosinophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5779, "association": 6.451193, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 14, "phenotype_reference": "Hematocrit (BLOO20) ", "LDSC Intercept": "1.0104 (0.024)", "LDSC Lambda GC": 1.2103, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "code": "BLOO20", "category": "g: blood", "phenotype": "Hematocrit", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.47, "association": 17.971876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0221 (0.0511)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 15, "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "LDSC Intercept": "1.0265 (0.0434)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "code": "BLOO21", "category": "g: blood", "phenotype": "High light scatter percentage of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 21.465454, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0417 (0.0682)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 16, "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "LDSC Intercept": "1.0267 (0.0431)", "LDSC Lambda GC": 1.2201, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "code": "BLOO22", "category": "g: blood", "phenotype": "High light scatter reticulocyte count", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6418, "association": 18.662701000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0416 (0.0672)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 17, "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "LDSC Intercept": "0.9869 (0.0181)", "LDSC Lambda GC": 1.0988, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "code": "BLOO25", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.2926, "association": 47.241421, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 18, "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Intercept": "1.0092 (0.0408)", "LDSC Lambda GC": 1.194, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "code": "BLOO26", "category": "g: blood", "phenotype": "Mean corpuscular hemoglobin MCH", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.7671, "association": 71.048289, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.012 (0.0532)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 19, "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Intercept": "1.0078 (0.0383)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "code": "BLOO27", "category": "g: blood", "phenotype": "Mean corpuscular volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.8174, "association": 61.090796999999995, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0096 (0.0469)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 20, "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Intercept": "1.0051 (0.0458)", "LDSC Lambda GC": 1.2005, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "code": "BLOO29", "category": "g: blood", "phenotype": "Mean platelet volume", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.901, "association": 5.105201999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0057 (0.0508)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 21, "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "LDSC Intercept": "0.961 (0.0204)", "LDSC Lambda GC": 1.2136, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "code": "BLOO30", "category": "g: blood", "phenotype": "Sum neutrophil eosinophil counts", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5234, "association": 13.26409, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 22, "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Intercept": "0.9571 (0.0217)", "LDSC Lambda GC": 1.1843, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "code": "BLOO31", "category": "g: blood", "phenotype": "Neutrophil percentage of granulocytes", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.5319, "association": 6.36227, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 23, "phenotype_reference": "Red cell distribution width (BLOO35) ", "LDSC Intercept": "0.9673 (0.0292)", "LDSC Lambda GC": 1.1651, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "code": "BLOO35", "category": "g: blood", "phenotype": "Red cell distribution width", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6273, "association": 43.032569, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "index": 24, "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "LDSC Intercept": "1.0125 (0.0416)", "LDSC Lambda GC": 1.2038, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "code": "BLOO36", "category": "g: blood", "phenotype": "Reticulocyte fraction of red cells", "PMID or link": "27863252", "LDSC Mean Chi-Square": 1.6369999999999998, "association": 31.610657, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0197 (0.0653)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 25, "phenotype_reference": "BMI (BODY08) ", "LDSC Intercept": "1.0353 (0.0162)", "LDSC Lambda GC": 2.0471, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "code": "BODY08", "category": "b: anthropometric", "phenotype": "BMI", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.6286, "association": 5.177427, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0217 (0.01)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26394269", "study size": 13239, "index": 26, "phenotype_reference": "Primary Biliary Cirrhosis (CIRR01) ", "LDSC Intercept": "1.0039 (0.0096)", "LDSC Lambda GC": 1.0496, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.3757 (0.0628)", "code": "CIRR01", "category": "e: autoimmune", "phenotype": "Primary Biliary Cirrhosis", "PMID or link": "26394269", "LDSC Mean Chi-Square": 1.1057, "association": 4.2222040000000005, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0369 (0.0909)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "index": 27, "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "LDSC Intercept": "0.979 (0.016)", "LDSC Lambda GC": 2.094, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "code": "EDUC03", "category": "c: mental and behavioural", "phenotype": "Educational Attainment: EduYears", "PMID or link": "30038396", "LDSC Mean Chi-Square": 2.6521, "association": 16.442985999999998, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 28, "phenotype_reference": "Fat mass (FATM04) ", "LDSC Intercept": "1.043 (0.0163)", "LDSC Lambda GC": 2.0175, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "code": "FATM04", "category": "b: anthropometric", "phenotype": "Fat mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 2.5512, "association": 15.835111999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0277 (0.0105)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "index": 29, "phenotype_reference": "Height 2014 (HEIG03) ", "LDSC Intercept": "1.2288 (0.0295)", "LDSC Lambda GC": 2.0007, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "code": "HEIG03", "category": "b: anthropometric", "phenotype": "Height 2014", "PMID or link": "25282103", "LDSC Mean Chi-Square": 2.9423, "association": 9.606494999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.1178 (0.0152)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "index": 30, "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Intercept": "0.7568 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "code": "HIPC01", "category": "b: anthropometric", "phenotype": "Hip circumference European", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1677, "association": 3.322129, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "index": 31, "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Intercept": "0.7536 (0.0085)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "code": "HIPC02", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1656, "association": 2.842876, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "index": 32, "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "LDSC Intercept": "0.8606 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "code": "HIPC03F", "category": "b: anthropometric", "phenotype": "Hip circumference European female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1134, "association": 4.590371, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "index": 33, "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "LDSC Intercept": "0.8672 (0.0096)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.204 (0.016)", "code": "HIPC03M", "category": "b: anthropometric", "phenotype": "Hip circumference European male BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1064, "association": 2.393909, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "index": 34, "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "LDSC Intercept": "0.8088 (0.0091)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "code": "HIPC03", "category": "b: anthropometric", "phenotype": "Hip circumference European BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2106, "association": 6.820649, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "index": 35, "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "LDSC Intercept": "0.8574 (0.0082)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "code": "HIPC04F", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1104, "association": 3.19256, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "index": 36, "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "LDSC Intercept": "0.8066 (0.009)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "code": "HIPC04", "category": "b: anthropometric", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.2088, "association": 4.246830999999999, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 37, "phenotype_reference": "Log10 Ferritin (IRON01) ", "LDSC Intercept": "1.0304 (0.0094)", "LDSC Lambda GC": 1.0618, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0388 (0.012)", "code": "IRON01", "category": "a: metabolic", "phenotype": "Log10 Ferritin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0704, "association": 3.4225199999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4321 (0.1331)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 38, "phenotype_reference": "Transferrin (IRON02) ", "LDSC Intercept": "1.0761 (0.0208)", "LDSC Lambda GC": 1.0864, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "code": "IRON02", "category": "a: metabolic", "phenotype": "Transferrin", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.1241, "association": 10.897806, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6135 (0.1678)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "index": 39, "phenotype_reference": "Transferrin saturation (IRON03) ", "LDSC Intercept": "1.0318 (0.0098)", "LDSC Lambda GC": 1.0436, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "code": "IRON03", "category": "a: metabolic", "phenotype": "Transferrin saturation", "PMID or link": "25352340", "LDSC Mean Chi-Square": 1.0484, "association": 10.406825, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.6579 (0.2019)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "nopmid.html", "study size": 353972, "index": 40, "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Intercept": "1.11 (0.0274)", "LDSC Lambda GC": 2.1851, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "code": "LEAN06", "category": "b: anthropometric", "phenotype": "Fat free mass / lean mass", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Mean Chi-Square": 3.1445, "association": 10.648997, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0513 (0.0128)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "index": 41, "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "LDSC Intercept": "1.1062 (0.0098)", "LDSC Lambda GC": 1.1715, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "code": "LUPU01", "category": "e: autoimmune", "phenotype": "Systemic Lupus Erythematosus", "PMID or link": "26502338", "LDSC Mean Chi-Square": 1.2181, "association": 26.835765999999996, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.4868 (0.0447)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "index": 42, "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "LDSC Intercept": "1.0058 (0.01)", "LDSC Lambda GC": 1.0557, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "code": "PRSC01", "category": "e: autoimmune", "phenotype": "Primary sclerosing cholangitis", "PMID or link": "27992413", "LDSC Mean Chi-Square": 1.0872, "association": 51.696437, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0664 (0.1142)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "study size": 58284, "index": 43, "phenotype_reference": "Rheumatoid Arthritis 2014 European (RHEU02) ", "LDSC Intercept": "0.9414 (0.0119)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1499 (0.0267)", "code": "RHEU02", "category": "h: musculoskeletal", "phenotype": "Rheumatoid Arthritis 2014 European", "PMID or link": "24390342", "LDSC Mean Chi-Square": 1.1164, "association": 5.926465, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "index": 44, "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Intercept": "1.033 (0.0104)", "LDSC Lambda GC": 1.5696, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "code": "SCHI01", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.755, "association": 16.426954000000002, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0437 (0.0138)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "index": 45, "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Intercept": "1.0382 (0.0103)", "LDSC Lambda GC": 1.5622, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "code": "SCHI02", "category": "c: mental and behavioural", "phenotype": "Schizophrenia PGC 2014 European", "PMID or link": "25056061", "LDSC Mean Chi-Square": 1.7409999999999999, "association": 19.882861, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "0.0515 (0.0139)", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "index": 46, "phenotype_reference": "Urate (URAT01) ", "LDSC Intercept": "0.8546 (0.045)", "LDSC Lambda GC": 1.0315, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "code": "URAT01", "category": "a: metabolic", "phenotype": "Urate", "PMID or link": "23263486", "LDSC Mean Chi-Square": 1.2383, "association": 4.716518, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}, {"phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "index": 47, "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "LDSC Intercept": "0.8244 (0.0084)", "LDSC Lambda GC": 1.0466, "gene": "ABT1", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "code": "WAIS03", "category": "b: anthropometric", "phenotype": "Waist circumference EU BMI-adjusted", "PMID or link": "25673412", "LDSC Mean Chi-Square": 1.1406, "association": 2.390779, "ENSEMBL": "ENSG00000146109", "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "variable": "Vagina", "value": 0.0}]}};
    var embed_opt = {"mode": "vega-lite"};

    function showError(el, error){
        el.innerHTML = ('<div class="error">'
                        + '<p>JavaScript Error: ' + error.message + '</p>'
                        + "<p>This usually means there's a typo in your chart specification. "
                        + "See the javascript console for the full traceback.</p>"
                        + '</div>');
        throw error;
    }
    const el = document.getElementById('vis');
    vegaEmbed("#vis", spec, embed_opt)
      .catch(error => showError(el, error));
  </script>
</body>
</html>
