
<!DOCTYPE html>

<html>
<head><style>.vega-actions a {margin-right: 12px;color: #757575;font-weight: normal;font-size: 13px;}.error {color: red;}</style><script src="https://cdn.jsdelivr.net/npm//vega@4" type="text/javascript"></script><script src="https://cdn.jsdelivr.net/npm//vega-lite@2.6.0" type="text/javascript"></script><script src="https://cdn.jsdelivr.net/npm//vega-embed@3" type="text/javascript"></script><meta content="text/html; charset=utf-8" http-equiv="Content-Type"/><meta content="IE=edge" http-equiv="X-UA-Compatible"/><meta content="width=device-width, initial-scale=1" name="viewport"/><link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/Font-Awesome-4.7/css/font-awesome.min.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/css/bootstrap.min.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/css/bootstrap-theme.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/images/gnomic_icon.png" rel="icon" type="image/png"/><script src="https://phenviz.navigome.com/js/jquery.min.js"></script><script src="https://phenviz.navigome.com/js/bootstrap.js"></script><title>Gene profile: ENSG00000168298</title></head>
<body>
<h4>Gene profile across phenotypes - ENSG00000168298 (HIST1H1E)</h4><div id="vis"></div><div class="col-lg-12 col-md-12 content-item content-item-1"><p>This is a visualization of GWAS-derived gene associations  across Navigome phenotypes. Only phenotypes for which the gene has at least one significant association (p-value â‰¤ 0.05/20000) in <a href="https://ctg.cncr.nl/software/magma">MAGMA</a> <b>and/or</b> <a href="https://github.com/hakyimlab/MetaXcan">S-PrediXcan</a> analyses are shown. The first panel is a visualization of gene-phenotype associations obtained using <a href="https://ctg.cncr.nl/software/magma">MAGMA</a> (-log10(p-value)). Hover over any bar to highlight the phenotype in the other panels. The other panels are parallel coordinates visualizations for tissue-wise associations (z-scores)computed using <a href="https://github.com/hakyimlab/MetaXcan">S-PrediXcan</a>  and eQTL expression data from GTEx and DGN. Significance thresholds are delineated with black lines. The parallel coordinates show which phenotypes have a significant association and in which tissues, based on gene expression data. Clicking on a specific phenotype will lead to the reference of the GWAS study.</p></div>
<script type="text/javascript">
    var spec = {"config": {"view": {"width": 400, "height": 300}}, "vconcat": [{"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "bar", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "href": {"type": "nominal", "field": "phenotype_link"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "quantitative", "axis": {"title": "MAGMA -log10(p)"}, "field": "value"}, "y": {"type": "nominal", "axis": {"title": ""}, "field": "phenotype_reference"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-4e93ee1a91df87c61b7d73e211f0f4d1"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "x": {"type": "quantitative", "field": "ThresholdVal"}}}], "title": "MAGMA gene-wise association"}], "center": true, "data": {"name": "data-3e1fb065cd8781767093314764a0e3fd"}}, {"vconcat": [{"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3800281253, 26.3800281253]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3800281253, 26.3800281253]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3800281253, 26.3800281253]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3800281253, 26.3800281253]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Nervous system", "width": 500}], "center": true, "data": {"name": "data-efad82c58a699a084a9027166ff7f256"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3800281253, 26.3800281253]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3800281253, 26.3800281253]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3800281253, 26.3800281253]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3800281253, 26.3800281253]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Digestive System", "width": 500}], "center": true, "data": {"name": "data-35bc6aa5225fc5d2dedfea2e1deb7be3"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3800281253, 26.3800281253]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3800281253, 26.3800281253]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3800281253, 26.3800281253]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3800281253, 26.3800281253]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Blood, Arteries and Heart", "width": 500}], "center": true, "data": {"name": "data-89120ac5ee49444cf0c1cf1ceab441ea"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3800281253, 26.3800281253]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3800281253, 26.3800281253]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3800281253, 26.3800281253]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3800281253, 26.3800281253]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Adipose Tissue, Muscle, Skin and Cells", "width": 500}], "center": true, "data": {"name": "data-f082e16b68550f89f6c163afa28a60cb"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3800281253, 26.3800281253]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3800281253, 26.3800281253]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3800281253, 26.3800281253]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-26.3800281253, 26.3800281253]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Endocrine System and Others", "width": 500}], "center": true, "data": {"name": "data-2bd4e0585dedf9b28037b8bc2f1839d0"}}]}], "spacing": 40, "$schema": "https://vega.github.io/schema/vega-lite/v2.6.0.json", "datasets": {"data-3e1fb065cd8781767093314764a0e3fd": [{"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "magma_log10p", "value": 22.26099}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "magma_log10p", "value": 9.554753999999999}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "magma_log10p", "value": 19.587623}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "magma_log10p", "value": 51.201874}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "magma_log10p", "value": 42.407013}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "magma_log10p", "value": 13.762507000000001}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "magma_log10p", "value": 8.895991}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "magma_log10p", "value": 7.0938}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "magma_log10p", "value": 8.468572}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "magma_log10p", "value": 21.2718}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "magma_log10p", "value": 17.853996}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "magma_log10p", "value": 9.688352}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "magma_log10p", "value": 24.084141}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "magma_log10p", "value": 29.67714}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "magma_log10p", "value": 29.385693}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "magma_log10p", "value": 5.915209}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "magma_log10p", "value": 63.195036}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "magma_log10p", "value": 85.801563}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "magma_log10p", "value": 91.283112}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "magma_log10p", "value": 4.865026}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "magma_log10p", "value": 6.1798839999999995}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "magma_log10p", "value": 5.959398}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "magma_log10p", "value": 5.092347}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "magma_log10p", "value": 47.538171000000006}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "magma_log10p", "value": 22.564681}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "magma_log10p", "value": 9.127721000000001}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "magma_log10p", "value": 7.567785000000001}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "magma_log10p", "value": 9.889141}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "magma_log10p", "value": 26.182369}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "magma_log10p", "value": 14.347967}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "magma_log10p", "value": 9.312542}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "magma_log10p", "value": 24.378056}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "magma_log10p", "value": 9.148736}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "magma_log10p", "value": 6.204982}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "magma_log10p", "value": 8.59779}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "magma_log10p", "value": 6.634736999999999}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "magma_log10p", "value": 10.398418}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "magma_log10p", "value": 7.987458}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "magma_log10p", "value": 14.576229999999999}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "magma_log10p", "value": 9.623714999999999}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "magma_log10p", "value": 7.159173}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "magma_log10p", "value": 12.597739}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "magma_log10p", "value": 14.707235}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "magma_log10p", "value": 25.026775}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "magma_log10p", "value": 21.791774}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "magma_log10p", "value": 17.280245999999998}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "magma_log10p", "value": 10.562852000000001}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "magma_log10p", "value": 7.1430240000000005}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "magma_log10p", "value": 8.124591}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "magma_log10p", "value": 41.355158}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "magma_log10p", "value": 6.102422}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "magma_log10p", "value": 21.859995}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "magma_log10p", "value": 22.385651}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "magma_log10p", "value": 12.042776}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "magma_log10p", "value": 9.588953}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "magma_log10p", "value": 9.706372}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "magma_log10p", "value": 9.520382000000001}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "magma_log10p", "value": 7.602477}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "magma_log10p", "value": 7.058459}], "data-4e93ee1a91df87c61b7d73e211f0f4d1": [{"Threshold": "MAGMA -log10(p) \u2265 5.60206", "ThresholdVal": 5.60206}], "data-efad82c58a699a084a9027166ff7f256": [{"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Brain_Amygdala", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Brain_Caudate", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Brain_Putamen", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Brain_Cerebellum", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Brain_Cortex", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Brain_Frontal_Cortex_BA9", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Brain_Hippocampus", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Brain_Hypothalamus", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Brain_Substantia_nigra", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Nerve_Tibial", "value": 2.66954996728}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Nerve_Tibial", "value": -2.96462639676}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Nerve_Tibial", "value": 1.9278774285299998}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Nerve_Tibial", "value": -3.9605154128099995}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Nerve_Tibial", "value": -2.79369431198}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Nerve_Tibial", "value": -1.51869409655}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Nerve_Tibial", "value": -1.04169858986}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Nerve_Tibial", "value": -3.18955304251}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Nerve_Tibial", "value": -3.37220833887}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Nerve_Tibial", "value": -2.63182967445}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Nerve_Tibial", "value": -3.8193441833699997}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Nerve_Tibial", "value": -1.3936296129199999}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Nerve_Tibial", "value": -2.3753227073599996}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Nerve_Tibial", "value": -2.07656301813}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Nerve_Tibial", "value": -1.7623852687000001}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Nerve_Tibial", "value": 0.217693220996}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Nerve_Tibial", "value": -4.975410703530001}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Nerve_Tibial", "value": -7.6597892184}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Nerve_Tibial", "value": -6.79990138566}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Nerve_Tibial", "value": 1.91530071021}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Nerve_Tibial", "value": -3.1055624249}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Nerve_Tibial", "value": -2.29211702365}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Nerve_Tibial", "value": 1.76150973384}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Nerve_Tibial", "value": 5.8225617214}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Nerve_Tibial", "value": -3.3133471040499995}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Nerve_Tibial", "value": 2.46336809836}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Nerve_Tibial", "value": 0.39755691218500006}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Nerve_Tibial", "value": 0.677636210517}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Nerve_Tibial", "value": 2.541817908}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Nerve_Tibial", "value": 0.516898870485}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Nerve_Tibial", "value": -1.19645679793}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Nerve_Tibial", "value": -0.613528919017}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Nerve_Tibial", "value": 0.233627075475}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Nerve_Tibial", "value": -0.11945771829700001}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Nerve_Tibial", "value": 0.454510422716}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Nerve_Tibial", "value": 0.21350049410900002}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Nerve_Tibial", "value": 2.47780300839}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Nerve_Tibial", "value": -0.0400035390204}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Nerve_Tibial", "value": 1.92185530931}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Nerve_Tibial", "value": 2.37272241222}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Nerve_Tibial", "value": -0.128312582467}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Nerve_Tibial", "value": 1.6786218095899998}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Nerve_Tibial", "value": 2.7525171358300002}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Nerve_Tibial", "value": -4.43306474175}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Nerve_Tibial", "value": 0.625512338153}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Nerve_Tibial", "value": 2.5531040035799997}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Nerve_Tibial", "value": 0.0122027720397}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Nerve_Tibial", "value": 0.465930581221}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Nerve_Tibial", "value": 0.07706950626180001}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Nerve_Tibial", "value": 3.53537153725}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Nerve_Tibial", "value": -1.9767490625400002}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Nerve_Tibial", "value": -3.12662190996}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Nerve_Tibial", "value": -3.1460205784599995}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Nerve_Tibial", "value": -0.950511224976}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Nerve_Tibial", "value": 1.44899241667}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Nerve_Tibial", "value": 2.3068275684099997}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Nerve_Tibial", "value": 1.5574295365900002}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Nerve_Tibial", "value": 2.30676444168}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Nerve_Tibial", "value": 1.1909926847399999}], "data-c4c9a654d45741ebc60aa567761bff19": [{"Text": "Significance Threshold", "Threshold": "PrediXcan zscore \u2265 4.70813", "ThresholdVal": 4.70813}], "data-4a3aa5bf8c5a41879455008930c25774": [{"Threshold": "PrediXcan zscore \u2264 -4.70813", "ThresholdVal": -4.70813}], "data-35bc6aa5225fc5d2dedfea2e1deb7be3": [{"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Colon_Sigmoid", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Colon_Transverse", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Esophagus_Mucosa", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Esophagus_Muscularis", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Liver", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Minor_Salivary_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Stomach", "value": -1.9072901370500002}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Stomach", "value": -6.3918560730400005}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Stomach", "value": -2.27387833086}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Stomach", "value": 17.1827804168}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Stomach", "value": 9.70462979695}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Stomach", "value": 6.56264087709}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Stomach", "value": 3.4957657687199997}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Stomach", "value": 3.8465986915699997}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Stomach", "value": 4.49936050579}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Stomach", "value": 5.73077811835}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Stomach", "value": 5.76463864993}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Stomach", "value": 3.60883446443}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Stomach", "value": 12.2816513802}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Stomach", "value": 5.77290596464}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Stomach", "value": 5.39470710477}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Stomach", "value": -2.44247691929}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Stomach", "value": 15.214066454600001}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Stomach", "value": 26.3800281253}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Stomach", "value": 22.6090184581}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Stomach", "value": 6.23688772431}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Stomach", "value": 3.86206603127}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Stomach", "value": -4.051087529}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Stomach", "value": 5.0107971567}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Stomach", "value": -20.3023519148}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Stomach", "value": 10.49281884}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Stomach", "value": -3.3787857238}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Stomach", "value": -0.0126951096374}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Stomach", "value": 0.51764722896}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Stomach", "value": -1.56988142783}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Stomach", "value": -4.37964040152}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Stomach", "value": 1.25941145017}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Stomach", "value": 3.0190424123}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Stomach", "value": 3.1399496084900003}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Stomach", "value": 0.850508840838}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Stomach", "value": 3.3460483035000004}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Stomach", "value": 3.11641181019}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Stomach", "value": 3.0033754304199998}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Stomach", "value": 1.7748576110299998}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Stomach", "value": 3.3329253518199997}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Stomach", "value": 2.8635417972400004}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Stomach", "value": 1.70148014833}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Stomach", "value": 3.13162638685}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Stomach", "value": -7.83532079218}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Stomach", "value": 12.2341666137}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Stomach", "value": 1.0091015662799998}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Stomach", "value": -5.97780949023}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Stomach", "value": 1.96224668642}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Stomach", "value": 1.0963904853}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Stomach", "value": 1.06645450861}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Stomach", "value": -8.36900104532}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Stomach", "value": 0.27566709144}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Stomach", "value": 2.57106514878}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Stomach", "value": 2.55353434935}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Stomach", "value": -4.03667231551}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Stomach", "value": 1.60401293227}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Stomach", "value": 2.27624863772}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Stomach", "value": 1.4489932188299999}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Stomach", "value": 2.18060861053}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Stomach", "value": 0.34460505691200005}], "data-89120ac5ee49444cf0c1cf1ceab441ea": [{"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "DGN_Whole_Blood", "value": -0.35156494829981105}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "DGN_Whole_Blood", "value": -0.4743279746881815}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "DGN_Whole_Blood", "value": -1.5106821403127382}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "DGN_Whole_Blood", "value": -1.2576975659785548}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "DGN_Whole_Blood", "value": -0.4776152597136363}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "DGN_Whole_Blood", "value": -1.137184529675482}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "DGN_Whole_Blood", "value": -3.107528881233084}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "DGN_Whole_Blood", "value": -2.196462539295419}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "DGN_Whole_Blood", "value": -2.1537090883864884}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "DGN_Whole_Blood", "value": -2.592920898172205}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "DGN_Whole_Blood", "value": -2.882503036667233}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "DGN_Whole_Blood", "value": -3.4251515953510183}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "DGN_Whole_Blood", "value": -0.9310286768552832}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "DGN_Whole_Blood", "value": 0.8586571313590559}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "DGN_Whole_Blood", "value": 0.5692216435795968}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "DGN_Whole_Blood", "value": 1.270411536809685}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "DGN_Whole_Blood", "value": -1.6074224172535492}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "DGN_Whole_Blood", "value": 0.3530035176139279}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "DGN_Whole_Blood", "value": 1.4333769587651255}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "DGN_Whole_Blood", "value": 2.309423847736566}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "DGN_Whole_Blood", "value": -2.143600505702654}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "DGN_Whole_Blood", "value": 0.9450325490816792}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "DGN_Whole_Blood", "value": 0.5407308860030221}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "DGN_Whole_Blood", "value": -0.25671604522176084}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "DGN_Whole_Blood", "value": 0.021829724231776818}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "DGN_Whole_Blood", "value": -0.1916267420193513}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "DGN_Whole_Blood", "value": -1.1465586886411292}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "DGN_Whole_Blood", "value": 1.3387649878430454}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "DGN_Whole_Blood", "value": -0.3196814243421463}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "DGN_Whole_Blood", "value": 0.4543402583528848}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "DGN_Whole_Blood", "value": -2.6308857140835533}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "DGN_Whole_Blood", "value": -0.7618808194570211}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "DGN_Whole_Blood", "value": -0.595975807463451}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "DGN_Whole_Blood", "value": -0.6038914999709429}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "DGN_Whole_Blood", "value": 0.6414392633075869}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "DGN_Whole_Blood", "value": 0.5117990841220015}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "DGN_Whole_Blood", "value": 0.1558248419883934}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "DGN_Whole_Blood", "value": -0.3005292829004824}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "DGN_Whole_Blood", "value": 0.08096101182056104}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "DGN_Whole_Blood", "value": 0.03589665024339122}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "DGN_Whole_Blood", "value": -0.43067600343521495}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "DGN_Whole_Blood", "value": -0.07706669052547492}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "DGN_Whole_Blood", "value": 1.4998542656866722}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "DGN_Whole_Blood", "value": -2.5431052409008683}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "DGN_Whole_Blood", "value": -0.1935573404419976}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "DGN_Whole_Blood", "value": 0.5269436702479559}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "DGN_Whole_Blood", "value": -0.1943367600397679}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "DGN_Whole_Blood", "value": 0.5611987782917393}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "DGN_Whole_Blood", "value": 0.8483976737926733}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "DGN_Whole_Blood", "value": 1.5248979593887813}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "DGN_Whole_Blood", "value": 0.3542025657194045}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "DGN_Whole_Blood", "value": -1.0206550953477742}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "DGN_Whole_Blood", "value": -0.9710985589819616}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "DGN_Whole_Blood", "value": -3.0781211721566377}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "DGN_Whole_Blood", "value": -0.7447470229834751}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "DGN_Whole_Blood", "value": -0.7820375016061805}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "DGN_Whole_Blood", "value": -0.6186214474231558}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "DGN_Whole_Blood", "value": -0.8484516625380236}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "DGN_Whole_Blood", "value": -1.4776360814632237}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Whole_Blood", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Artery_Aorta", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Artery_Coronary", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Artery_Tibial", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Heart_Left_Ventricle", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Spleen", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Spleen", "value": 0.0}], "data-f082e16b68550f89f6c163afa28a60cb": [{"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Adipose_Subcutaneous", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Muscle_Skeletal", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Cells_Transformed_fibroblasts", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Cells_EBV_transformed_lymphocytes", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.0}], "data-2bd4e0585dedf9b28037b8bc2f1839d0": [{"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Adrenal_Gland", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Pituitary", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Pancreas", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Thyroid", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Lung", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Ovary", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Prostate", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Testis", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Uterus", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5273, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0389 (0.0109)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0217, "index": 0, "code": "BFPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Body fat percentage complete", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "gene": "HIST1H1E", "LDSC Intercept": "1.0594 (0.0166)", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "association": 22.26099, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.768, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2531, "index": 1, "code": "BLOO01", "ENSEMBL": "ENSG00000168298", "phenotype": "platelet count", "LDSC Total Observed Scale h2": "0.2268 (0.0185)", "gene": "HIST1H1E", "LDSC Intercept": "0.9967 (0.0271)", "phenotype_reference": "Platelet count (BLOO01) ", "association": 9.554753999999999, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6473, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0032 (0.0394)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2464, "index": 2, "code": "BLOO02", "ENSEMBL": "ENSG00000168298", "phenotype": "red blood cell count", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "gene": "HIST1H1E", "LDSC Intercept": "1.0021 (0.0255)", "phenotype_reference": "Red blood cell count (BLOO02) ", "association": 19.587623, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4634, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0199 (0.0502)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 3, "code": "BLOO03", "ENSEMBL": "ENSG00000168298", "phenotype": "hemoglobin concentration", "LDSC Total Observed Scale h2": "0.1294 (0.0115)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0233)", "phenotype_reference": "Hemoglobin concentration (BLOO03) ", "association": 51.201874, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6377, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0243 (0.0638)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.207, "index": 4, "code": "BLOO04", "ENSEMBL": "ENSG00000168298", "phenotype": "reticulocyte count", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "gene": "HIST1H1E", "LDSC Intercept": "1.0155 (0.0407)", "phenotype_reference": "Reticulocyte count (BLOO04) ", "association": 42.407013, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6776, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0686 (0.0499)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 5, "code": "BLOO05", "ENSEMBL": "ENSG00000168298", "phenotype": "monocyte count", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0465 (0.0338)", "phenotype_reference": "Monocyte count (BLOO05) ", "association": 13.762507000000001, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6196, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1876, "index": 6, "code": "BLOO07", "ENSEMBL": "ENSG00000168298", "phenotype": "eosinophil count", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "gene": "HIST1H1E", "LDSC Intercept": "0.9587 (0.0273)", "phenotype_reference": "Eosinophil count (BLOO07) ", "association": 8.895991, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5217, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 7, "code": "BLOO09", "ENSEMBL": "ENSG00000168298", "phenotype": "granulocyte count", "LDSC Total Observed Scale h2": "0.1589 (0.0186)", "gene": "HIST1H1E", "LDSC Intercept": "0.9591 (0.0206)", "phenotype_reference": "Granulocyte count (BLOO09) ", "association": 7.0938, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5375, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2168, "index": 8, "code": "BLOO10", "ENSEMBL": "ENSG00000168298", "phenotype": "myeloid white cell count", "LDSC Total Observed Scale h2": "0.1625 (0.0179)", "gene": "HIST1H1E", "LDSC Intercept": "0.9623 (0.02)", "phenotype_reference": "Myeloid white cell count (BLOO10) ", "association": 8.468572, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6107, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 9, "code": "BLOO11", "ENSEMBL": "ENSG00000168298", "phenotype": "lymphocyte count", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "gene": "HIST1H1E", "LDSC Intercept": "0.9683 (0.0197)", "phenotype_reference": "Lymphocyte count (BLOO11) ", "association": 21.2718, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5935, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2431, "index": 10, "code": "BLOO12", "ENSEMBL": "ENSG00000168298", "phenotype": "white blood cell count", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "gene": "HIST1H1E", "LDSC Intercept": "0.9648 (0.0192)", "phenotype_reference": "White blood cell count (BLOO12) ", "association": 17.853996, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5912, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1908, "index": 11, "code": "BLOO16", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum eosinophil basophil counts", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "gene": "HIST1H1E", "LDSC Intercept": "0.9694 (0.0258)", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "association": 9.688352, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.47, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0221 (0.0511)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2103, "index": 12, "code": "BLOO20", "ENSEMBL": "ENSG00000168298", "phenotype": "Hematocrit", "LDSC Total Observed Scale h2": "0.1316 (0.0126)", "gene": "HIST1H1E", "LDSC Intercept": "1.0104 (0.024)", "phenotype_reference": "Hematocrit (BLOO20) ", "association": 24.084141, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0417 (0.0682)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 13, "code": "BLOO21", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter percentage of red cells", "LDSC Total Observed Scale h2": "0.1652 (0.0187)", "gene": "HIST1H1E", "LDSC Intercept": "1.0265 (0.0434)", "phenotype_reference": "High light scatter percentage of red cells (BLOO21) ", "association": 29.67714, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6418, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0416 (0.0672)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2201, "index": 14, "code": "BLOO22", "ENSEMBL": "ENSG00000168298", "phenotype": "High light scatter reticulocyte count", "LDSC Total Observed Scale h2": "0.1657 (0.0191)", "gene": "HIST1H1E", "LDSC Intercept": "1.0267 (0.0431)", "phenotype_reference": "High light scatter reticulocyte count (BLOO22) ", "association": 29.385693, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.4532, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0626 (0.0887)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1555, "index": 15, "code": "BLOO23", "ENSEMBL": "ENSG00000168298", "phenotype": "Immature fraction of reticulocytes", "LDSC Total Observed Scale h2": "0.1178 (0.0143)", "gene": "HIST1H1E", "LDSC Intercept": "1.0284 (0.0402)", "phenotype_reference": "Immature fraction of reticulocytes (BLOO23) ", "association": 5.915209, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2926, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.0988, "index": 16, "code": "BLOO25", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin concentration MCHC", "LDSC Total Observed Scale h2": "0.0921 (0.0103)", "gene": "HIST1H1E", "LDSC Intercept": "0.9869 (0.0181)", "phenotype_reference": "Mean corpuscular hemoglobin concentration MCHC (BLOO25) ", "association": 63.195036, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7671, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.012 (0.0532)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.194, "index": 17, "code": "BLOO26", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular hemoglobin MCH", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "gene": "HIST1H1E", "LDSC Intercept": "1.0092 (0.0408)", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "association": 85.801563, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.8174, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0096 (0.0469)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 18, "code": "BLOO27", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean corpuscular volume", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "gene": "HIST1H1E", "LDSC Intercept": "1.0078 (0.0383)", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "association": 91.283112, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.901, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0057 (0.0508)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2005, "index": 19, "code": "BLOO29", "ENSEMBL": "ENSG00000168298", "phenotype": "Mean platelet volume", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "gene": "HIST1H1E", "LDSC Intercept": "1.0051 (0.0458)", "phenotype_reference": "Mean platelet volume (BLOO29) ", "association": 4.865026, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5234, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2136, "index": 20, "code": "BLOO30", "ENSEMBL": "ENSG00000168298", "phenotype": "Sum neutrophil eosinophil counts", "LDSC Total Observed Scale h2": "0.1591 (0.0182)", "gene": "HIST1H1E", "LDSC Intercept": "0.961 (0.0204)", "phenotype_reference": "Sum neutrophil eosinophil counts (BLOO30) ", "association": 6.1798839999999995, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6946, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0253 (0.0354)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2498, "index": 21, "code": "BLOO33", "ENSEMBL": "ENSG00000168298", "phenotype": "Plateletcrit", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "1.0176 (0.0246)", "phenotype_reference": "Plateletcrit (BLOO33) ", "association": 5.959398, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.5949, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1747, "index": 22, "code": "BLOO34", "ENSEMBL": "ENSG00000168298", "phenotype": "Platelet distribution width", "LDSC Total Observed Scale h2": "0.1674 (0.0209)", "gene": "HIST1H1E", "LDSC Intercept": "0.9902 (0.0289)", "phenotype_reference": "Platelet distribution width (BLOO34) ", "association": 5.092347, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6273, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.1651, "index": 23, "code": "BLOO35", "ENSEMBL": "ENSG00000168298", "phenotype": "Red cell distribution width", "LDSC Total Observed Scale h2": "0.1899 (0.0231)", "gene": "HIST1H1E", "LDSC Intercept": "0.9673 (0.0292)", "phenotype_reference": "Red cell distribution width (BLOO35) ", "association": 47.538171000000006, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6369999999999998, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "study size": 173480, "LDSC Ratio": "0.0197 (0.0653)", "category": "g: blood", "PMID or link": "27863252", "LDSC Lambda GC": 1.2038, "index": 24, "code": "BLOO36", "ENSEMBL": "ENSG00000168298", "phenotype": "Reticulocyte fraction of red cells", "LDSC Total Observed Scale h2": "0.1699 (0.0202)", "gene": "HIST1H1E", "LDSC Intercept": "1.0125 (0.0416)", "phenotype_reference": "Reticulocyte fraction of red cells (BLOO36) ", "association": 22.564681, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6286, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0217 (0.01)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0471, "index": 25, "code": "BODY08", "ENSEMBL": "ENSG00000168298", "phenotype": "BMI", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0353 (0.0162)", "phenotype_reference": "BMI (BODY08) ", "association": 9.127721000000001, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3682, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27046643", "study size": 111114, "LDSC Ratio": "0.0875 (0.0232)", "category": "c: mental and behavioural", "PMID or link": "27046643", "LDSC Lambda GC": 1.3068, "index": 26, "code": "EDUC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Attainment of a college or a university degree", "LDSC Total Observed Scale h2": "0.1536 (0.0079)", "gene": "HIST1H1E", "LDSC Intercept": "1.0322 (0.0085)", "phenotype_reference": "Attainment of a college or a university degree (EDUC02) ", "association": 7.567785000000001, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 2.6521, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/30038396", "study size": 766345, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "c: mental and behavioural", "PMID or link": "30038396", "LDSC Lambda GC": 2.094, "index": 27, "code": "EDUC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Educational Attainment: EduYears", "LDSC Total Observed Scale h2": "0.1123 (0.003)", "gene": "HIST1H1E", "LDSC Intercept": "0.979 (0.016)", "phenotype_reference": "Educational Attainment: EduYears (EDUC03) ", "association": 9.889141, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 2.5512, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0277 (0.0105)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.0175, "index": 28, "code": "FATM04", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat mass", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "gene": "HIST1H1E", "LDSC Intercept": "1.043 (0.0163)", "phenotype_reference": "Fat mass (FATM04) ", "association": 26.182369, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0661, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28898252", "study size": 123665, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "28898252", "LDSC Lambda GC": 1.0105, "index": 29, "code": "GLYC22", "ENSEMBL": "ENSG00000168298", "phenotype": "HbA1c European", "LDSC Total Observed Scale h2": "0.0609 (0.0072)", "gene": "HIST1H1E", "LDSC Intercept": "0.9192 (0.0088)", "phenotype_reference": "HbA1c European (GLYC22) ", "association": 14.347967, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2268, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23563607", "study size": 16196, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23563607", "LDSC Lambda GC": 1.0466, "index": 30, "code": "HEIG02", "ENSEMBL": "ENSG00000168298", "phenotype": "Extreme Height", "LDSC Total Observed Scale h2": "1.0377 (0.0906)", "gene": "HIST1H1E", "LDSC Intercept": "0.8886 (0.0149)", "phenotype_reference": "Extreme Height (HEIG02) ", "association": 9.312542, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 2.9423, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25282103", "study size": 253288, "LDSC Ratio": "0.1178 (0.0152)", "category": "b: anthropometric", "PMID or link": "25282103", "LDSC Lambda GC": 2.0007, "index": 31, "code": "HEIG03", "ENSEMBL": "ENSG00000168298", "phenotype": "Height 2014", "LDSC Total Observed Scale h2": "0.341 (0.0174)", "gene": "HIST1H1E", "LDSC Intercept": "1.2288 (0.0295)", "phenotype_reference": "Height 2014 (HEIG03) ", "association": 24.378056, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1891, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 73137, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 32, "code": "HEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Height female 2013", "LDSC Total Observed Scale h2": "0.2477 (0.0171)", "gene": "HIST1H1E", "LDSC Intercept": "0.8259 (0.0107)", "phenotype_reference": "Height female 2013 (HEIG04F) ", "association": 9.148736, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1464, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 60586, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 1.0466, "index": 33, "code": "HEIG04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Height male 2013", "LDSC Total Observed Scale h2": "0.2593 (0.017)", "gene": "HIST1H1E", "LDSC Intercept": "0.8343 (0.0098)", "phenotype_reference": "Height male 2013 (HEIG04M) ", "association": 6.204982, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1677, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145308, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 34, "code": "HIPC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "gene": "HIST1H1E", "LDSC Intercept": "0.7568 (0.0085)", "phenotype_reference": "Hip circumference European (HIPC01) ", "association": 8.59779, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1656, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 145414, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 35, "code": "HIPC02", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "gene": "HIST1H1E", "LDSC Intercept": "0.7536 (0.0085)", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "association": 6.634736999999999, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1134, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86792, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 36, "code": "HIPC03F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1549 (0.0119)", "gene": "HIST1H1E", "LDSC Intercept": "0.8606 (0.0084)", "phenotype_reference": "Hip circumference European female BMI-adjusted (HIPC03F) ", "association": 10.398418, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1064, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56939, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 37, "code": "HIPC03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European male BMI-adjusted", "LDSC Total Observed Scale h2": "0.204 (0.016)", "gene": "HIST1H1E", "LDSC Intercept": "0.8672 (0.0096)", "phenotype_reference": "Hip circumference European male BMI-adjusted (HIPC03M) ", "association": 7.987458, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2106, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143579, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 38, "code": "HIPC03", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference European BMI-adjusted", "LDSC Total Observed Scale h2": "0.1459 (0.009)", "gene": "HIST1H1E", "LDSC Intercept": "0.8088 (0.0091)", "phenotype_reference": "Hip circumference European BMI-adjusted (HIPC03) ", "association": 14.576229999999999, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1104, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 86647, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 39, "code": "HIPC04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries female BMI-adjusted", "LDSC Total Observed Scale h2": "0.1546 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "0.8574 (0.0082)", "phenotype_reference": "Hip circumference All Ancestries female BMI-adjusted (HIPC04F) ", "association": 9.623714999999999, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1051, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 56978, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 40, "code": "HIPC04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.2033 (0.0159)", "gene": "HIST1H1E", "LDSC Intercept": "0.866 (0.0095)", "phenotype_reference": "Hip circumference All Ancestries male BMI-adjusted (HIPC04M) ", "association": 7.159173, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2088, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 143575, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 41, "code": "HIPC04", "ENSEMBL": "ENSG00000168298", "phenotype": "Hip circumference All Ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1457 (0.0089)", "gene": "HIST1H1E", "LDSC Intercept": "0.8066 (0.009)", "phenotype_reference": "Hip circumference All Ancestries BMI-adjusted (HIPC04) ", "association": 12.597739, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1241, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6135 (0.1678)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0864, "index": 42, "code": "IRON02", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin", "LDSC Total Observed Scale h2": "0.0471 (0.019)", "gene": "HIST1H1E", "LDSC Intercept": "1.0761 (0.0208)", "phenotype_reference": "Transferrin (IRON02) ", "association": 14.707235, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0484, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25352340", "study size": 48972, "LDSC Ratio": "0.6579 (0.2019)", "category": "a: metabolic", "PMID or link": "25352340", "LDSC Lambda GC": 1.0436, "index": 43, "code": "IRON03", "ENSEMBL": "ENSG00000168298", "phenotype": "Transferrin saturation", "LDSC Total Observed Scale h2": "0.0159 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "1.0318 (0.0098)", "phenotype_reference": "Transferrin saturation (IRON03) ", "association": 25.026775, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 3.1445, "phenotype_link": "nopmid.html", "study size": 353972, "LDSC Ratio": "0.0513 (0.0128)", "category": "b: anthropometric", "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "LDSC Lambda GC": 2.1851, "index": 44, "code": "LEAN06", "ENSEMBL": "ENSG00000168298", "phenotype": "Fat free mass / lean mass", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "gene": "HIST1H1E", "LDSC Intercept": "1.11 (0.0274)", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "association": 21.791774, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2181, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/26502338", "study size": 23210, "LDSC Ratio": "0.4868 (0.0447)", "category": "e: autoimmune", "PMID or link": "26502338", "LDSC Lambda GC": 1.1715, "index": 45, "code": "LUPU01", "ENSEMBL": "ENSG00000168298", "phenotype": "Systemic Lupus Erythematosus", "LDSC Total Observed Scale h2": "0.2352 (0.0409)", "gene": "HIST1H1E", "LDSC Intercept": "1.1062 (0.0098)", "phenotype_reference": "Systemic Lupus Erythematosus (LUPU01) ", "association": 17.280245999999998, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.6669999999999998, "phenotype_link": "nopmid.html", "study size": 287535, "LDSC Ratio": "0.0079 (0.0184)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.4745, "index": 46, "code": "NEUR02B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete meta-analysis", "LDSC Total Observed Scale h2": "0.1126 (0.006)", "gene": "HIST1H1E", "LDSC Intercept": "1.0053 (0.0123)", "phenotype_reference": "Neuroticism complete meta-analysis (NEUR02B) ", "association": 10.562852000000001, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3388, "phenotype_link": "nopmid.html", "study size": 142875, "LDSC Ratio": "0.0199 (0.0314)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2697, "index": 47, "code": "NEUR02M", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism complete male", "LDSC Total Observed Scale h2": "0.1145 (0.0091)", "gene": "HIST1H1E", "LDSC Intercept": "1.0067 (0.0106)", "phenotype_reference": "Neuroticism complete male (NEUR02M) ", "association": 7.1430240000000005, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3272, "phenotype_link": "nopmid.html", "study size": 157355, "LDSC Ratio": "0.0111 (0.0334)", "category": "c: mental and behavioural", "PMID or link": "Not published (J. Coleman 2017 UK Biobank)", "LDSC Lambda GC": 1.2531, "index": 48, "code": "NEUR03B", "ENSEMBL": "ENSG00000168298", "phenotype": "Neuroticism healthy meta-analysis", "LDSC Total Observed Scale h2": "0.1008 (0.0081)", "gene": "HIST1H1E", "LDSC Intercept": "1.0036 (0.0109)", "phenotype_reference": "Neuroticism healthy meta-analysis (NEUR03B) ", "association": 8.124591, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0872, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27992413", "study size": 24751, "LDSC Ratio": "0.0664 (0.1142)", "category": "e: autoimmune", "PMID or link": "27992413", "LDSC Lambda GC": 1.0557, "index": 49, "code": "PRSC01", "ENSEMBL": "ENSG00000168298", "phenotype": "Primary sclerosing cholangitis", "LDSC Total Observed Scale h2": "0.154 (0.0346)", "gene": "HIST1H1E", "LDSC Intercept": "1.0058 (0.01)", "phenotype_reference": "Primary sclerosing cholangitis (PRSC01) ", "association": 41.355158, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.3127, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24280982", "study size": 39202, "LDSC Ratio": "0.0568 (0.0274)", "category": "c: mental and behavioural", "PMID or link": "24280982", "LDSC Lambda GC": 1.2664, "index": 50, "code": "SCBP01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia+bipolar", "LDSC Total Observed Scale h2": "0.3793 (0.0217)", "gene": "HIST1H1E", "LDSC Intercept": "1.0178 (0.0086)", "phenotype_reference": "Schizophrenia+bipolar (SCBP01) ", "association": 6.102422, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.755, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 82315, "LDSC Ratio": "0.0437 (0.0138)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5696, "index": 51, "code": "SCHI01", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "gene": "HIST1H1E", "LDSC Intercept": "1.033 (0.0104)", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "association": 21.859995, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.7409999999999999, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "study size": 77096, "LDSC Ratio": "0.0515 (0.0139)", "category": "c: mental and behavioural", "PMID or link": "25056061", "LDSC Lambda GC": 1.5622, "index": 52, "code": "SCHI02", "ENSEMBL": "ENSG00000168298", "phenotype": "Schizophrenia PGC 2014 European", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "gene": "HIST1H1E", "LDSC Intercept": "1.0382 (0.0103)", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "association": 22.385651, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.2383, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23263486", "study size": 110347, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "a: metabolic", "PMID or link": "23263486", "LDSC Lambda GC": 1.0315, "index": 53, "code": "URAT01", "ENSEMBL": "ENSG00000168298", "phenotype": "Urate", "LDSC Total Observed Scale h2": "0.1522 (0.053)", "gene": "HIST1H1E", "LDSC Intercept": "0.8546 (0.045)", "phenotype_reference": "Urate (URAT01) ", "association": 12.042776, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0873, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 60393, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 54, "code": "WAIS03M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU male BMI-adjusted", "LDSC Total Observed Scale h2": "0.177 (0.0123)", "gene": "HIST1H1E", "LDSC Intercept": "0.8744 (0.008)", "phenotype_reference": "Waist circumference EU male BMI-adjusted (WAIS03M) ", "association": 9.588953, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1406, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 229089, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 55, "code": "WAIS03", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference EU BMI-adjusted", "LDSC Total Observed Scale h2": "0.1087 (0.0069)", "gene": "HIST1H1E", "LDSC Intercept": "0.8244 (0.0084)", "phenotype_reference": "Waist circumference EU BMI-adjusted (WAIS03) ", "association": 9.706372, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.084, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 109077, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 56, "code": "WAIS04M", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries male BMI-adjusted", "LDSC Total Observed Scale h2": "0.1741 (0.0121)", "gene": "HIST1H1E", "LDSC Intercept": "0.8729 (0.0079)", "phenotype_reference": "Waist circumference all ancestries male BMI-adjusted (WAIS04M) ", "association": 9.520382000000001, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.1420000000000001, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "study size": 153478, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "25673412", "LDSC Lambda GC": 1.0466, "index": 57, "code": "WAIS04", "ENSEMBL": "ENSG00000168298", "phenotype": "Waist circumference all ancestries BMI-adjusted", "LDSC Total Observed Scale h2": "0.1095 (0.007)", "gene": "HIST1H1E", "LDSC Intercept": "0.8232 (0.0084)", "phenotype_reference": "Waist circumference all ancestries BMI-adjusted (WAIS04) ", "association": 7.602477, "variable": "Vagina", "value": 0.0}, {"LDSC Mean Chi-Square": 1.0711, "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/23754948", "study size": 67592, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "category": "b: anthropometric", "PMID or link": "23754948", "LDSC Lambda GC": 0.9986, "index": 58, "code": "WEIG04F", "ENSEMBL": "ENSG00000168298", "phenotype": "Weight European female", "LDSC Total Observed Scale h2": "0.188 (0.0107)", "gene": "HIST1H1E", "LDSC Intercept": "0.8241 (0.0071)", "phenotype_reference": "Weight European female (WEIG04F) ", "association": 7.058459, "variable": "Vagina", "value": 0.0}]}};
    var embed_opt = {"mode": "vega-lite"};

    function showError(el, error){
        el.innerHTML = ('<div class="error">'
                        + '<p>JavaScript Error: ' + error.message + '</p>'
                        + "<p>This usually means there's a typo in your chart specification. "
                        + "See the javascript console for the full traceback.</p>"
                        + '</div>');
        throw error;
    }
    const el = document.getElementById('vis');
    vegaEmbed("#vis", spec, embed_opt)
      .catch(error => showError(el, error));
  </script>
</body>
</html>
