
<!DOCTYPE html>

<html>
<head><style>.vega-actions a {margin-right: 12px;color: #757575;font-weight: normal;font-size: 13px;}.error {color: red;}</style><script src="https://cdn.jsdelivr.net/npm//vega@4" type="text/javascript"></script><script src="https://cdn.jsdelivr.net/npm//vega-lite@2.6.0" type="text/javascript"></script><script src="https://cdn.jsdelivr.net/npm//vega-embed@3" type="text/javascript"></script><meta content="text/html; charset=utf-8" http-equiv="Content-Type"/><meta content="IE=edge" http-equiv="X-UA-Compatible"/><meta content="width=device-width, initial-scale=1" name="viewport"/><link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/Font-Awesome-4.7/css/font-awesome.min.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/css/bootstrap.min.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/css/bootstrap-theme.css" rel="stylesheet" type="text/css"/><link href="https://phenviz.navigome.com/static/images/gnomic_icon.png" rel="icon" type="image/png"/><script src="https://phenviz.navigome.com/js/jquery.min.js"></script><script src="https://phenviz.navigome.com/js/bootstrap.js"></script><title>Gene profile: ENSG00000102882</title></head>
<body>
<h4>Gene profile across phenotypes - ENSG00000102882 (MAPK3)</h4><div id="vis"></div><div class="col-lg-12 col-md-12 content-item content-item-1"><p>This is a visualization of GWAS-derived gene associations  across Navigome phenotypes. Only phenotypes for which the gene has at least one significant association (p-value â‰¤ 0.05/20000) in <a href="https://ctg.cncr.nl/software/magma">MAGMA</a> <b>and/or</b> <a href="https://github.com/hakyimlab/MetaXcan">S-PrediXcan</a> analyses are shown. The first panel is a visualization of gene-phenotype associations obtained using <a href="https://ctg.cncr.nl/software/magma">MAGMA</a> (-log10(p-value)). Hover over any bar to highlight the phenotype in the other panels. The other panels are parallel coordinates visualizations for tissue-wise associations (z-scores)computed using <a href="https://github.com/hakyimlab/MetaXcan">S-PrediXcan</a>  and eQTL expression data from GTEx and DGN. Significance thresholds are delineated with black lines. The parallel coordinates show which phenotypes have a significant association and in which tissues, based on gene expression data. Clicking on a specific phenotype will lead to the reference of the GWAS study.</p></div>
<script type="text/javascript">
    var spec = {"config": {"view": {"width": 400, "height": 300}}, "vconcat": [{"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "bar", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "href": {"type": "nominal", "field": "phenotype_link"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "quantitative", "axis": {"title": "MAGMA -log10(p)"}, "field": "value"}, "y": {"type": "nominal", "axis": {"title": ""}, "field": "phenotype_reference"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-4e93ee1a91df87c61b7d73e211f0f4d1"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "x": {"type": "quantitative", "field": "ThresholdVal"}}}], "title": "MAGMA gene-wise association"}], "center": true, "data": {"name": "data-eb56f0ed0accdff1fa4632023e8943c2"}}, {"vconcat": [{"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-11.7438473929, 11.7438473929]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-11.7438473929, 11.7438473929]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-11.7438473929, 11.7438473929]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-11.7438473929, 11.7438473929]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Nervous system", "width": 500}], "center": true, "data": {"name": "data-363c419029372cefa9ddac46d0be8808"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-11.7438473929, 11.7438473929]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-11.7438473929, 11.7438473929]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-11.7438473929, 11.7438473929]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-11.7438473929, 11.7438473929]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Digestive System", "width": 500}], "center": true, "data": {"name": "data-1adfdf1fe82dac4b73cf14e5234a3cac"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-11.7438473929, 11.7438473929]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-11.7438473929, 11.7438473929]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-11.7438473929, 11.7438473929]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-11.7438473929, 11.7438473929]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Blood, Arteries and Heart", "width": 500}], "center": true, "data": {"name": "data-abc9f6ee66e6d8bf5a0f6fc9706c2fe4"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-11.7438473929, 11.7438473929]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-11.7438473929, 11.7438473929]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-11.7438473929, 11.7438473929]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-11.7438473929, 11.7438473929]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Adipose Tissue, Muscle, Skin and Cells", "width": 500}], "center": true, "data": {"name": "data-7aec11b198cc19b6e399868038710187"}}, {"hconcat": [{"mark": "square", "encoding": {"color": {"condition": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitourinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}, "selection": "selector002"}, "value": "lightgray"}, "opacity": {"value": 1}, "size": {"value": 100}, "y": {"type": "nominal", "axis": {"orient": "left", "title": "Click on a category"}, "field": "category"}}, "selection": {"selector002": {"type": "multi", "fields": ["category"], "empty": "all"}}}, {"layer": [{"mark": "line", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"condition": {"value": 0.7, "selection": {"not": "selector001"}}, "value": 1}, "size": {"condition": {"value": 1.5, "selection": {"not": "selector001"}}, "value": 4}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-11.7438473929, 11.7438473929]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": "circle", "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "opacity": {"value": 0}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-11.7438473929, 11.7438473929]}, "title": "S-PrediXcan z-score"}}, "selection": {"selector001": {"type": "single", "empty": "none", "on": "mouseover", "fields": ["phenotype_reference"]}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"mark": {"type": "text", "align": "center", "baseline": "middle", "color": "black", "dy": -7}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "text": {"type": "nominal", "field": "code"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-11.7438473929, 11.7438473929]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}, {"filter": "(abs(datum.value) >= '4.70813')"}, {"filter": {"selection": "selector001"}}]}, {"mark": {"type": "circle", "opacity": 0}, "encoding": {"color": {"type": "nominal", "field": "category", "legend": null, "scale": {"domain": ["a: metabolic", "b: anthropometric", "c: mental and behavioural", "d: nervous system", "e: autoimmune", "f: genitorurinary", "g: blood", "h: musculoskeletal", "i: aging", "j: other"]}}, "detail": {"type": "quantitative", "field": "index"}, "href": {"type": "nominal", "field": "phenotype_link"}, "tooltip": [{"type": "nominal", "field": "variable"}, {"type": "quantitative", "field": "value"}, {"type": "quantitative", "field": "study size"}, {"type": "nominal", "field": "category"}, {"type": "nominal", "field": "code"}, {"type": "nominal", "field": "phenotype"}, {"type": "nominal", "field": "PMID or link"}, {"type": "nominal", "field": "LDSC Total Observed Scale h2"}, {"type": "quantitative", "field": "LDSC Lambda GC"}, {"type": "quantitative", "field": "LDSC Mean Chi-Square"}, {"type": "nominal", "field": "LDSC Intercept"}, {"type": "nominal", "field": "LDSC Ratio"}, {"type": "nominal", "field": "phenotype_reference"}, {"type": "nominal", "field": "phenotype_link"}], "x": {"type": "nominal", "axis": {"labelAngle": -45}, "field": "variable", "title": ""}, "y": {"type": "quantitative", "field": "value", "scale": {"domain": [-11.7438473929, 11.7438473929]}, "title": "S-PrediXcan z-score"}}, "transform": [{"filter": {"selection": "selector002"}}]}, {"data": {"name": "data-c4c9a654d45741ebc60aa567761bff19"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}, {"data": {"name": "data-4a3aa5bf8c5a41879455008930c25774"}, "mark": {"type": "rule", "color": "black", "size": 1}, "encoding": {"tooltip": [{"type": "nominal", "field": "Threshold"}], "y": {"type": "quantitative", "field": "ThresholdVal"}}}], "height": 250, "title": "Endocrine System and Others", "width": 500}], "center": true, "data": {"name": "data-ad432fa7542fc08fd16eb5dccb2a1caa"}}]}], "spacing": 40, "$schema": "https://vega.github.io/schema/vega-lite/v2.6.0.json", "datasets": {"data-eb56f0ed0accdff1fa4632023e8943c2": [{"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "magma_log10p", "value": 8.629691000000001}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "magma_log10p", "value": 26.359171000000003}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "magma_log10p", "value": 8.615539}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "magma_log10p", "value": 9.509171}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "magma_log10p", "value": 16.578823999999997}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "magma_log10p", "value": 13.748922}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "magma_log10p", "value": 8.634924}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "magma_log10p", "value": 13.947151999999999}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "magma_log10p", "value": 13.717808999999999}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "magma_log10p", "value": 13.344372}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "magma_log10p", "value": 1.576984}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "magma_log10p", "value": 31.679729}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "magma_log10p", "value": 29.107065000000002}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "magma_log10p", "value": 5.640544}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "magma_log10p", "value": 11.473299}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "magma_log10p", "value": 5.5066120000000005}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "magma_log10p", "value": 5.971794999999999}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "magma_log10p", "value": 5.510056}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "magma_log10p", "value": 22.787253}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "magma_log10p", "value": 16.593238}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "magma_log10p", "value": 7.560936}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "magma_log10p", "value": 7.435796000000001}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "magma_log10p", "value": 8.208561999999999}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "magma_log10p", "value": 8.572774}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "magma_log10p", "value": 40.661524}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "magma_log10p", "value": 11.08008}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "magma_log10p", "value": 6.460459999999999}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "magma_log10p", "value": 6.178978}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "magma_log10p", "value": 6.2500160000000005}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "magma_log10p", "value": 6.277325}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "magma_log10p", "value": 5.881405}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "magma_log10p", "value": 7.295000999999999}], "data-4e93ee1a91df87c61b7d73e211f0f4d1": [{"Threshold": "MAGMA -log10(p) \u2265 5.60206", "ThresholdVal": 5.60206}], "data-363c419029372cefa9ddac46d0be8808": [{"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Brain_Amygdala", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Brain_Anterior_cingulate_cortex_BA24", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Brain_Caudate", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Brain_Nucleus_accumbens", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Brain_Putamen", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Brain_Cerebellar_Hemisphere", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Brain_Cerebellum", "value": -3.93359444676}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Brain_Cerebellum", "value": 8.083844522569999}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Brain_Cerebellum", "value": 3.7038955862300003}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Brain_Cerebellum", "value": 3.3758391802200003}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Brain_Cerebellum", "value": 5.163514801730001}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Brain_Cerebellum", "value": 3.76374811512}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Brain_Cerebellum", "value": 2.5031141087}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Brain_Cerebellum", "value": 4.95624508328}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Brain_Cerebellum", "value": 4.33657557842}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Brain_Cerebellum", "value": 4.3420152247}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Brain_Cerebellum", "value": 1.95348725051}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Brain_Cerebellum", "value": -9.582710647339999}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Brain_Cerebellum", "value": -9.5567175466}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Brain_Cerebellum", "value": 1.53846398409}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Brain_Cerebellum", "value": -4.27601680356}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Brain_Cerebellum", "value": 2.16840775798}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Brain_Cerebellum", "value": -2.9994105430599998}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Brain_Cerebellum", "value": -3.1445066021}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Brain_Cerebellum", "value": -6.331294165549999}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Brain_Cerebellum", "value": -5.03982641255}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Brain_Cerebellum", "value": -4.11243780858}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Brain_Cerebellum", "value": -3.9210813872300005}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Brain_Cerebellum", "value": -4.00578338918}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Brain_Cerebellum", "value": -3.81586368491}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Brain_Cerebellum", "value": -6.157429907}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Brain_Cerebellum", "value": 3.87436982987}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Brain_Cerebellum", "value": -2.81845217643}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Brain_Cerebellum", "value": 2.2685485339500002}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Brain_Cerebellum", "value": 2.56056269239}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Brain_Cerebellum", "value": -3.6630822731199997}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Brain_Cerebellum", "value": -3.2737356441000003}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Brain_Cerebellum", "value": -3.5554350097099996}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Brain_Cortex", "value": -4.07954668929}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Brain_Cortex", "value": 8.949066044630001}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Brain_Cortex", "value": 4.1097603619}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Brain_Cortex", "value": 3.9156738867300005}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Brain_Cortex", "value": 5.022259890369999}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Brain_Cortex", "value": 4.86354698297}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Brain_Cortex", "value": 3.6025951730699997}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Brain_Cortex", "value": 4.65815868276}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Brain_Cortex", "value": 3.76936792916}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Brain_Cortex", "value": 3.76431941371}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Brain_Cortex", "value": 2.14693471253}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Brain_Cortex", "value": -9.72657885792}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Brain_Cortex", "value": -9.554311714389998}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Brain_Cortex", "value": 3.22678154075}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Brain_Cortex", "value": -3.5521386488499997}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Brain_Cortex", "value": 3.0264287044}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Brain_Cortex", "value": -2.71470534875}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Brain_Cortex", "value": -2.8842377381800004}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Brain_Cortex", "value": -6.2902761091}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Brain_Cortex", "value": -5.37524732102}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Brain_Cortex", "value": -4.15621117666}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Brain_Cortex", "value": -4.26212335406}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Brain_Cortex", "value": -4.03667971753}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Brain_Cortex", "value": -4.14947082839}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Brain_Cortex", "value": -6.9223471848199996}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Brain_Cortex", "value": 4.17719888298}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Brain_Cortex", "value": -3.0592248082999998}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Brain_Cortex", "value": 2.7586848844200005}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Brain_Cortex", "value": 2.95210768999}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Brain_Cortex", "value": -3.53822602922}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Brain_Cortex", "value": -3.08204162703}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Brain_Cortex", "value": -3.4516516140099998}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Brain_Frontal_Cortex_BA9", "value": -4.44139708504}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Brain_Frontal_Cortex_BA9", "value": 8.793030591960001}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Brain_Frontal_Cortex_BA9", "value": 4.8211011815300004}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Brain_Frontal_Cortex_BA9", "value": 3.9363588640300007}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Brain_Frontal_Cortex_BA9", "value": 5.604859660630001}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Brain_Frontal_Cortex_BA9", "value": 3.66949773692}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Brain_Frontal_Cortex_BA9", "value": 2.5453907340599997}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Brain_Frontal_Cortex_BA9", "value": 5.158311469719999}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Brain_Frontal_Cortex_BA9", "value": 4.74221226148}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Brain_Frontal_Cortex_BA9", "value": 4.76539758711}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Brain_Frontal_Cortex_BA9", "value": 1.81762772037}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Brain_Frontal_Cortex_BA9", "value": -9.961386571639999}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Brain_Frontal_Cortex_BA9", "value": -9.67801057609}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Brain_Frontal_Cortex_BA9", "value": 4.48090977556}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Brain_Frontal_Cortex_BA9", "value": -4.47126080554}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Brain_Frontal_Cortex_BA9", "value": 2.7886308562999997}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Brain_Frontal_Cortex_BA9", "value": -2.6432620535900004}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Brain_Frontal_Cortex_BA9", "value": -2.81252644905}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Brain_Frontal_Cortex_BA9", "value": -6.465078344019999}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Brain_Frontal_Cortex_BA9", "value": -5.634519043609999}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Brain_Frontal_Cortex_BA9", "value": -3.5052168266900003}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Brain_Frontal_Cortex_BA9", "value": -3.32543103998}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Brain_Frontal_Cortex_BA9", "value": -3.41712393948}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Brain_Frontal_Cortex_BA9", "value": -3.2343793802599996}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Brain_Frontal_Cortex_BA9", "value": -6.933607587010001}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Brain_Frontal_Cortex_BA9", "value": 5.11217573621}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Brain_Frontal_Cortex_BA9", "value": -2.39408086783}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Brain_Frontal_Cortex_BA9", "value": 3.45537329921}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Brain_Frontal_Cortex_BA9", "value": 3.6856315416199994}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Brain_Frontal_Cortex_BA9", "value": -3.0834516455000003}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Brain_Frontal_Cortex_BA9", "value": -2.7511549024699997}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Brain_Frontal_Cortex_BA9", "value": -2.9853433386400003}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Brain_Hippocampus", "value": -2.36754372674}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Brain_Hippocampus", "value": 6.31887845299}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Brain_Hippocampus", "value": 2.48861794823}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Brain_Hippocampus", "value": 2.5714570670900003}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Brain_Hippocampus", "value": 3.1585147563700002}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Brain_Hippocampus", "value": 5.781310877590001}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Brain_Hippocampus", "value": 3.63995910281}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Brain_Hippocampus", "value": 2.83315369214}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Brain_Hippocampus", "value": 1.93954648288}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Brain_Hippocampus", "value": 2.1024490791900003}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Brain_Hippocampus", "value": 3.05871065227}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Brain_Hippocampus", "value": -7.652769739660001}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Brain_Hippocampus", "value": -6.893621732660001}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Brain_Hippocampus", "value": 3.2217622777}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Brain_Hippocampus", "value": -1.8519247892}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Brain_Hippocampus", "value": 2.7191108057099997}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Brain_Hippocampus", "value": -1.22845364388}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Brain_Hippocampus", "value": -1.25023265002}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Brain_Hippocampus", "value": -3.3213878742799996}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Brain_Hippocampus", "value": -2.35389608322}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Brain_Hippocampus", "value": -2.17115037542}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Brain_Hippocampus", "value": -2.4775572844}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Brain_Hippocampus", "value": -2.1175148126299996}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Brain_Hippocampus", "value": -2.41432703794}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Brain_Hippocampus", "value": -1.80257623433}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Brain_Hippocampus", "value": 1.76861377367}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Brain_Hippocampus", "value": -1.8270109282499998}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Brain_Hippocampus", "value": 1.22156614942}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Brain_Hippocampus", "value": 1.3982490075100003}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Brain_Hippocampus", "value": -2.09578422713}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Brain_Hippocampus", "value": -1.3909502409799999}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Brain_Hippocampus", "value": -2.1186546120400003}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Brain_Hypothalamus", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Brain_Spinal_cord_cervical_C1", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Brain_Substantia_nigra", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Nerve_Tibial", "value": -4.8865256003}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Nerve_Tibial", "value": 9.29219389042}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Nerve_Tibial", "value": 5.09563397551}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Nerve_Tibial", "value": 4.163136467159999}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Nerve_Tibial", "value": 6.151146441259999}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Nerve_Tibial", "value": 2.78153052217}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Nerve_Tibial", "value": 2.8080597236}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Nerve_Tibial", "value": 5.50470177502}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Nerve_Tibial", "value": 5.24103711932}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Nerve_Tibial", "value": 5.07737754666}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Nerve_Tibial", "value": 0.559241417891}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Nerve_Tibial", "value": -10.7700596327}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Nerve_Tibial", "value": -10.2988108877}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Nerve_Tibial", "value": 4.138295086499999}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Nerve_Tibial", "value": -4.52977276313}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Nerve_Tibial", "value": 3.75572941747}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Nerve_Tibial", "value": -3.4458893230500003}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Nerve_Tibial", "value": -3.5548516634400005}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Nerve_Tibial", "value": -6.98497357735}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Nerve_Tibial", "value": -6.318967533130001}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Nerve_Tibial", "value": -4.250669135340001}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Nerve_Tibial", "value": -4.249551875919999}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Nerve_Tibial", "value": -4.140885402059999}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Nerve_Tibial", "value": -4.13399065692}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Nerve_Tibial", "value": -8.7190537214}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Nerve_Tibial", "value": 5.766130527740001}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Nerve_Tibial", "value": -2.06869146201}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Nerve_Tibial", "value": 3.93756035738}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Nerve_Tibial", "value": 4.070898512009999}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Nerve_Tibial", "value": -3.8687122233399998}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Nerve_Tibial", "value": -3.4435966408099996}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Nerve_Tibial", "value": -3.7507166658499997}], "data-c4c9a654d45741ebc60aa567761bff19": [{"Text": "Significance Threshold", "Threshold": "PrediXcan zscore \u2265 4.70813", "ThresholdVal": 4.70813}], "data-4a3aa5bf8c5a41879455008930c25774": [{"Threshold": "PrediXcan zscore \u2264 -4.70813", "ThresholdVal": -4.70813}], "data-1adfdf1fe82dac4b73cf14e5234a3cac": [{"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Colon_Sigmoid", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Colon_Transverse", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Esophagus_Gastroesophageal_Junction", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Esophagus_Mucosa", "value": -2.78943603386}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Esophagus_Mucosa", "value": 2.85598574842}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Esophagus_Mucosa", "value": -1.22624732628}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Esophagus_Mucosa", "value": 3.12894633479}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Esophagus_Mucosa", "value": 0.910624475897}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Esophagus_Mucosa", "value": 10.3479797612}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Esophagus_Mucosa", "value": 5.869414889880001}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Esophagus_Mucosa", "value": 0.914304098218}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Esophagus_Mucosa", "value": -1.2318505659299999}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Esophagus_Mucosa", "value": -0.31607530211999996}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Esophagus_Mucosa", "value": 6.10204714101}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Esophagus_Mucosa", "value": -3.6489589694}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Esophagus_Mucosa", "value": -2.37748117848}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Esophagus_Mucosa", "value": 2.52244883751}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Esophagus_Mucosa", "value": 0.41959457958400004}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Esophagus_Mucosa", "value": 2.2764967517700003}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Esophagus_Mucosa", "value": -2.80438391828}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Esophagus_Mucosa", "value": -2.55577132569}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Esophagus_Mucosa", "value": -3.80277932667}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Esophagus_Mucosa", "value": -2.9673492779400004}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Esophagus_Mucosa", "value": -2.51158994067}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Esophagus_Mucosa", "value": -3.86103392441}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Esophagus_Mucosa", "value": -2.7420464711}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Esophagus_Mucosa", "value": -3.8735927870899998}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Esophagus_Mucosa", "value": -2.11423105033}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Esophagus_Mucosa", "value": 3.6368709859400004}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Esophagus_Mucosa", "value": -1.5200387192200002}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Esophagus_Mucosa", "value": 2.9775899383599995}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Esophagus_Mucosa", "value": 3.10007909575}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Esophagus_Mucosa", "value": -3.08064187395}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Esophagus_Mucosa", "value": -2.17825813471}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Esophagus_Mucosa", "value": -3.2990390169}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Esophagus_Muscularis", "value": -4.26122588373}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Esophagus_Muscularis", "value": 8.89556290008}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Esophagus_Muscularis", "value": 4.620815698}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Esophagus_Muscularis", "value": 4.91068501856}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Esophagus_Muscularis", "value": 7.84072520858}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Esophagus_Muscularis", "value": 2.4450610039}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Esophagus_Muscularis", "value": 2.6108590559000002}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Esophagus_Muscularis", "value": 7.16541666356}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Esophagus_Muscularis", "value": 6.985299555680001}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Esophagus_Muscularis", "value": 6.87368755416}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Esophagus_Muscularis", "value": 0.391970956691}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Esophagus_Muscularis", "value": -10.3839693346}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Esophagus_Muscularis", "value": -10.2655673075}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Esophagus_Muscularis", "value": 4.0550970485699995}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Esophagus_Muscularis", "value": -6.2772749157700005}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Esophagus_Muscularis", "value": 4.63125744313}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Esophagus_Muscularis", "value": -3.9464426506400003}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Esophagus_Muscularis", "value": -4.09169989565}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Esophagus_Muscularis", "value": -6.51252251926}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Esophagus_Muscularis", "value": -6.49187692472}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Esophagus_Muscularis", "value": -5.33027819287}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Esophagus_Muscularis", "value": -5.0239308950699995}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Esophagus_Muscularis", "value": -5.19631784976}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Esophagus_Muscularis", "value": -4.88637352954}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Esophagus_Muscularis", "value": -10.0541191279}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Esophagus_Muscularis", "value": 6.65398766204}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Esophagus_Muscularis", "value": -1.68252009788}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Esophagus_Muscularis", "value": 3.70078626995}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Esophagus_Muscularis", "value": 3.6778486642400003}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Esophagus_Muscularis", "value": -4.24788940966}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Esophagus_Muscularis", "value": -4.1421962170699995}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Esophagus_Muscularis", "value": -4.112731324049999}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Liver", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Liver", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Liver", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Liver", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Liver", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Liver", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Liver", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Liver", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Liver", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Liver", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Liver", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Liver", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Liver", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Liver", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Liver", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Liver", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Liver", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Liver", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Liver", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Liver", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Liver", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Liver", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Liver", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Liver", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Liver", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Liver", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Liver", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Liver", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Liver", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Liver", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Liver", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Liver", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Minor_Salivary_Gland", "value": -4.89931936441}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Minor_Salivary_Gland", "value": 8.86711300242}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Minor_Salivary_Gland", "value": 4.67203137488}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Minor_Salivary_Gland", "value": 5.02153891604}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Minor_Salivary_Gland", "value": 6.94247603887}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Minor_Salivary_Gland", "value": 5.71554050011}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Minor_Salivary_Gland", "value": 4.3247646497000005}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Minor_Salivary_Gland", "value": 6.579043206540001}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Minor_Salivary_Gland", "value": 5.4918964674}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Minor_Salivary_Gland", "value": 5.46675509008}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Minor_Salivary_Gland", "value": 2.34015859713}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Minor_Salivary_Gland", "value": -10.1180394839}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Minor_Salivary_Gland", "value": -10.2293606187}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Minor_Salivary_Gland", "value": 3.23498304505}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Minor_Salivary_Gland", "value": -5.21015737666}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Minor_Salivary_Gland", "value": 4.29983831583}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Minor_Salivary_Gland", "value": -3.71403184519}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Minor_Salivary_Gland", "value": -3.9228463924400003}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Minor_Salivary_Gland", "value": -8.10250430283}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Minor_Salivary_Gland", "value": -7.295522340700001}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Minor_Salivary_Gland", "value": -4.8940815894}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Minor_Salivary_Gland", "value": -5.30097077709}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Minor_Salivary_Gland", "value": -4.77108364378}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Minor_Salivary_Gland", "value": -5.15582793388}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Minor_Salivary_Gland", "value": -9.83324212041}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Minor_Salivary_Gland", "value": 5.13176298491}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Minor_Salivary_Gland", "value": -3.97970795631}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Minor_Salivary_Gland", "value": 3.9048362554500002}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Minor_Salivary_Gland", "value": 4.00580072051}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Minor_Salivary_Gland", "value": -4.94285539667}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Minor_Salivary_Gland", "value": -4.0726310221499995}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Minor_Salivary_Gland", "value": -4.7855850594}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Small_Intestine_Terminal_Ileum", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Stomach", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Stomach", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Stomach", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Stomach", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Stomach", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Stomach", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Stomach", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Stomach", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Stomach", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Stomach", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Stomach", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Stomach", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Stomach", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Stomach", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Stomach", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Stomach", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Stomach", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Stomach", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Stomach", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Stomach", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Stomach", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Stomach", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Stomach", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Stomach", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Stomach", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Stomach", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Stomach", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Stomach", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Stomach", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Stomach", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Stomach", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Stomach", "value": 0.0}], "data-abc9f6ee66e6d8bf5a0f6fc9706c2fe4": [{"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "DGN_Whole_Blood", "value": -5.526169515619475}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "DGN_Whole_Blood", "value": 10.299107535100957}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "DGN_Whole_Blood", "value": 5.871408865860984}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "DGN_Whole_Blood", "value": 5.8508383566478575}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "DGN_Whole_Blood", "value": 8.188064616551733}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "DGN_Whole_Blood", "value": 4.720529603351063}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "DGN_Whole_Blood", "value": 4.2257342168373535}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "DGN_Whole_Blood", "value": 7.671187622041184}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "DGN_Whole_Blood", "value": 6.858413088099232}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "DGN_Whole_Blood", "value": 6.690007523978196}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "DGN_Whole_Blood", "value": 1.2458966833888705}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "DGN_Whole_Blood", "value": -11.43503503896445}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "DGN_Whole_Blood", "value": -11.348273573400512}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "DGN_Whole_Blood", "value": 3.8840521543680944}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "DGN_Whole_Blood", "value": -6.248214841706757}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "DGN_Whole_Blood", "value": 5.301367586859891}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "DGN_Whole_Blood", "value": -4.6067266000616165}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "DGN_Whole_Blood", "value": -4.900800187974653}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "DGN_Whole_Blood", "value": -8.551564231216513}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "DGN_Whole_Blood", "value": -8.005800737568531}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "DGN_Whole_Blood", "value": -5.306881099488409}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "DGN_Whole_Blood", "value": -5.762488173000974}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "DGN_Whole_Blood", "value": -5.1874645869766844}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "DGN_Whole_Blood", "value": -5.619593935737907}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "DGN_Whole_Blood", "value": -11.134481531422749}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "DGN_Whole_Blood", "value": 6.837187892230389}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "DGN_Whole_Blood", "value": -2.744335155466866}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "DGN_Whole_Blood", "value": 4.6886749573948245}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "DGN_Whole_Blood", "value": 4.731797579371495}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "DGN_Whole_Blood", "value": -5.2543576232417095}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "DGN_Whole_Blood", "value": -4.487726740865004}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "DGN_Whole_Blood", "value": -5.103265159329805}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Whole_Blood", "value": -6.07057150468}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Whole_Blood", "value": 9.38277477504}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Whole_Blood", "value": 5.30790016298}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Whole_Blood", "value": 5.010318539130001}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Whole_Blood", "value": 7.39660652462}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Whole_Blood", "value": 4.4125249663699995}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Whole_Blood", "value": 3.5278198409300003}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Whole_Blood", "value": 6.89131038379}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Whole_Blood", "value": 6.22664668916}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Whole_Blood", "value": 6.12827061605}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Whole_Blood", "value": 1.50990954617}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Whole_Blood", "value": -10.9535039281}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Whole_Blood", "value": -10.7090393041}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Whole_Blood", "value": 4.04493793966}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Whole_Blood", "value": -5.6876002343300005}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Whole_Blood", "value": 4.4315362806300005}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Whole_Blood", "value": -4.68394000439}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Whole_Blood", "value": -4.9519979964}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Whole_Blood", "value": -9.23602179188}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Whole_Blood", "value": -8.58274867497}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Whole_Blood", "value": -5.3179856964}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Whole_Blood", "value": -5.731373731590001}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Whole_Blood", "value": -5.18713488389}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Whole_Blood", "value": -5.5752153796399995}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Whole_Blood", "value": -11.7438473929}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Whole_Blood", "value": 6.62223766858}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Whole_Blood", "value": -3.16095994167}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Whole_Blood", "value": 5.40856763669}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Whole_Blood", "value": 5.44382668727}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Whole_Blood", "value": -5.30262320477}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Whole_Blood", "value": -4.50866535153}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Whole_Blood", "value": -5.13733781626}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Artery_Aorta", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Artery_Coronary", "value": -1.5201144833000002}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Artery_Coronary", "value": 2.7691879516299998}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Artery_Coronary", "value": 2.2116853586}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Artery_Coronary", "value": 2.41145031477}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Artery_Coronary", "value": 3.86997201736}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Artery_Coronary", "value": 1.2791179212100001}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Artery_Coronary", "value": 1.12291003987}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Artery_Coronary", "value": 3.8217962694699996}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Artery_Coronary", "value": 3.56371674328}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Artery_Coronary", "value": 3.6322241350199995}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Artery_Coronary", "value": 0.24227580683400002}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Artery_Coronary", "value": -3.30155794408}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Artery_Coronary", "value": -3.5236891894199998}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Artery_Coronary", "value": 3.18588696114}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Artery_Coronary", "value": -3.5386854003699995}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Artery_Coronary", "value": 4.25583445928}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Artery_Coronary", "value": -2.62586126701}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Artery_Coronary", "value": -2.72202656683}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Artery_Coronary", "value": -2.7353080521599997}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Artery_Coronary", "value": -3.1659762540700003}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Artery_Coronary", "value": -1.3183654072700002}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Artery_Coronary", "value": -2.6002626973500003}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Artery_Coronary", "value": -1.28523226698}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Artery_Coronary", "value": -2.52906636928}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Artery_Coronary", "value": -5.87563430935}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Artery_Coronary", "value": 2.37559938757}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Artery_Coronary", "value": -1.02448943855}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Artery_Coronary", "value": 1.24727783168}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Artery_Coronary", "value": 1.18038751895}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Artery_Coronary", "value": -2.11451428065}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Artery_Coronary", "value": -0.966841013116}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Artery_Coronary", "value": -2.0472352779}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Artery_Tibial", "value": -5.39497657299}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Artery_Tibial", "value": 9.31421720704}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Artery_Tibial", "value": 4.62186368241}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Artery_Tibial", "value": 4.58766032938}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Artery_Tibial", "value": 6.90076169796}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Artery_Tibial", "value": 4.263869064430001}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Artery_Tibial", "value": 3.6608263977199997}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Artery_Tibial", "value": 6.53630916505}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Artery_Tibial", "value": 5.62956906086}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Artery_Tibial", "value": 5.629918568559999}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Artery_Tibial", "value": 1.40186437236}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Artery_Tibial", "value": -10.4801757389}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Artery_Tibial", "value": -10.1392651584}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Artery_Tibial", "value": 3.8625257857}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Artery_Tibial", "value": -5.3409392571}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Artery_Tibial", "value": 3.3541621617199997}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Artery_Tibial", "value": -3.54314030242}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Artery_Tibial", "value": -3.70332665415}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Artery_Tibial", "value": -7.80505810533}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Artery_Tibial", "value": -7.1562937163399996}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Artery_Tibial", "value": -4.33958975231}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Artery_Tibial", "value": -4.21871419602}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Artery_Tibial", "value": -4.241123522830001}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Artery_Tibial", "value": -4.1018214143}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Artery_Tibial", "value": -8.95271017296}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Artery_Tibial", "value": 5.05321538601}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Artery_Tibial", "value": -3.2033065579400004}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Artery_Tibial", "value": 3.08298720078}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Artery_Tibial", "value": 3.19248090728}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Artery_Tibial", "value": -3.77780989631}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Artery_Tibial", "value": -3.41087495639}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Artery_Tibial", "value": -3.64886439385}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Heart_Atrial_Appendage", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Heart_Left_Ventricle", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Spleen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Spleen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Spleen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Spleen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Spleen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Spleen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Spleen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Spleen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Spleen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Spleen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Spleen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Spleen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Spleen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Spleen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Spleen", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Spleen", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Spleen", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Spleen", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Spleen", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Spleen", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Spleen", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Spleen", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Spleen", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Spleen", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Spleen", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Spleen", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Spleen", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Spleen", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Spleen", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Spleen", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Spleen", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Spleen", "value": 0.0}], "data-7aec11b198cc19b6e399868038710187": [{"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Adipose_Subcutaneous", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Adipose_Visceral_Omentum", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Breast_Mammary_Tissue", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Muscle_Skeletal", "value": -4.57453352482}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Muscle_Skeletal", "value": 9.46597736647}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Muscle_Skeletal", "value": 4.75135732754}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Muscle_Skeletal", "value": 4.316705446369999}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Muscle_Skeletal", "value": 5.99505497179}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Muscle_Skeletal", "value": 4.37388084351}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Muscle_Skeletal", "value": 3.03150083269}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Muscle_Skeletal", "value": 5.503454190759999}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Muscle_Skeletal", "value": 4.93026196198}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Muscle_Skeletal", "value": 4.95177291866}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Muscle_Skeletal", "value": 2.10848280072}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Muscle_Skeletal", "value": -10.5806104443}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Muscle_Skeletal", "value": -10.433274816}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Muscle_Skeletal", "value": 3.99478787535}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Muscle_Skeletal", "value": -4.67555676269}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Muscle_Skeletal", "value": 3.55039964434}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Muscle_Skeletal", "value": -3.8022622387199996}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Muscle_Skeletal", "value": -4.02968075345}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Muscle_Skeletal", "value": -6.885190957260001}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Muscle_Skeletal", "value": -6.092556141959999}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Muscle_Skeletal", "value": -4.67420569836}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Muscle_Skeletal", "value": -4.79514840562}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Muscle_Skeletal", "value": -4.55673377898}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Muscle_Skeletal", "value": -4.6638550936900005}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Muscle_Skeletal", "value": -7.66226503538}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Muscle_Skeletal", "value": 5.34161244322}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Muscle_Skeletal", "value": -2.9638512238800003}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Muscle_Skeletal", "value": 3.22615455598}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Muscle_Skeletal", "value": 3.44800834355}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Muscle_Skeletal", "value": -4.313741053}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Muscle_Skeletal", "value": -3.6978708451499998}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Muscle_Skeletal", "value": -4.17648783597}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Cells_Transformed_fibroblasts", "value": -6.13735754391}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Cells_Transformed_fibroblasts", "value": 8.78994514305}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Cells_Transformed_fibroblasts", "value": 5.75435666584}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Cells_Transformed_fibroblasts", "value": 3.9783517302699996}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Cells_Transformed_fibroblasts", "value": 6.24111584284}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Cells_Transformed_fibroblasts", "value": 3.33479329763}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Cells_Transformed_fibroblasts", "value": 2.68570479589}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Cells_Transformed_fibroblasts", "value": 5.4751728333}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Cells_Transformed_fibroblasts", "value": 5.3049349781099995}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Cells_Transformed_fibroblasts", "value": 5.15331552131}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Cells_Transformed_fibroblasts", "value": 1.0384332725799998}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Cells_Transformed_fibroblasts", "value": -10.078523695}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Cells_Transformed_fibroblasts", "value": -9.87022277727}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Cells_Transformed_fibroblasts", "value": 3.64719568885}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Cells_Transformed_fibroblasts", "value": -4.46657023385}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Cells_Transformed_fibroblasts", "value": 3.42865150259}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Cells_Transformed_fibroblasts", "value": -4.5497050445}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Cells_Transformed_fibroblasts", "value": -4.82017853037}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Cells_Transformed_fibroblasts", "value": -8.3556477144}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Cells_Transformed_fibroblasts", "value": -8.11564489295}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Cells_Transformed_fibroblasts", "value": -4.239847430869999}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Cells_Transformed_fibroblasts", "value": -4.88120481552}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Cells_Transformed_fibroblasts", "value": -4.132803121459999}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Cells_Transformed_fibroblasts", "value": -4.74611062915}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Cells_Transformed_fibroblasts", "value": -10.5116131552}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Cells_Transformed_fibroblasts", "value": 6.4196492434}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Cells_Transformed_fibroblasts", "value": -1.65675820493}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Cells_Transformed_fibroblasts", "value": 5.2142808258999995}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Cells_Transformed_fibroblasts", "value": 5.195450651730001}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Cells_Transformed_fibroblasts", "value": -4.36701228711}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Cells_Transformed_fibroblasts", "value": -3.3749175956800004}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Cells_Transformed_fibroblasts", "value": -4.2271459701}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Cells_EBV_transformed_lymphocytes", "value": -3.8479119900199996}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 8.869443605939999}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 4.98987614593}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 4.2525490885}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 6.29370342127}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 4.19290674146}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 3.3078806995499996}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 5.87675726594}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 5.24608206962}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 5.17658485982}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 1.62837218715}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Cells_EBV_transformed_lymphocytes", "value": -9.90759461697}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Cells_EBV_transformed_lymphocytes", "value": -9.732766274089999}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 3.6234303367000003}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Cells_EBV_transformed_lymphocytes", "value": -4.86519209428}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 4.00667787273}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Cells_EBV_transformed_lymphocytes", "value": -3.04032706737}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Cells_EBV_transformed_lymphocytes", "value": -3.21138351569}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Cells_EBV_transformed_lymphocytes", "value": -6.7786129396899995}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Cells_EBV_transformed_lymphocytes", "value": -5.773375656240001}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Cells_EBV_transformed_lymphocytes", "value": -4.34090345017}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Cells_EBV_transformed_lymphocytes", "value": -4.47427226926}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Cells_EBV_transformed_lymphocytes", "value": -4.2121393043}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Cells_EBV_transformed_lymphocytes", "value": -4.36453492304}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Cells_EBV_transformed_lymphocytes", "value": -8.74107897068}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 4.82120138602}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Cells_EBV_transformed_lymphocytes", "value": -3.8719392376199995}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 3.2819488928099996}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Cells_EBV_transformed_lymphocytes", "value": 3.26451382524}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Cells_EBV_transformed_lymphocytes", "value": -3.9796191345}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Cells_EBV_transformed_lymphocytes", "value": -3.2897958344}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Cells_EBV_transformed_lymphocytes", "value": -3.90604958345}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Skin_Not_Sun_Exposed_Suprapubic", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -5.023663162}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 9.54967920742}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 5.412498252869999}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 4.533107698349999}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 6.85046371339}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.03485982009}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 2.7868280900499998}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 6.396019294480001}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 5.90744756654}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 5.7707536911599995}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 0.847429925451}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -10.6520248236}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -10.4734847317}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 4.05571439988}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -5.446643205780001}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 4.08370208921}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -3.7950228246300006}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -4.0224100044}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -7.25940696079}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -6.88420558565}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -4.61388156987}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -4.54981032336}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -4.50883869069}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -4.42457941112}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -9.296240044580001}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 5.97822119071}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -2.72741426382}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.7557146378699993}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": 3.9378002451999996}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -4.08987030558}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -3.7387138387400003}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Skin_Sun_Exposed_Lower_leg", "value": -3.9492653944400002}], "data-ad432fa7542fc08fd16eb5dccb2a1caa": [{"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Adrenal_Gland", "value": -0.7266359933050001}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Adrenal_Gland", "value": 3.67908172595}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Adrenal_Gland", "value": 0.983064209811}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Adrenal_Gland", "value": 1.32651664705}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Adrenal_Gland", "value": 0.382844863054}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Adrenal_Gland", "value": 4.95185723436}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Adrenal_Gland", "value": 2.5178571729400003}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Adrenal_Gland", "value": 0.497789372577}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Adrenal_Gland", "value": -0.5213765907069999}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Adrenal_Gland", "value": -0.20278849064800003}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Adrenal_Gland", "value": 3.2102985122099996}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Adrenal_Gland", "value": -4.06637632607}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Adrenal_Gland", "value": -3.34214262291}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Adrenal_Gland", "value": 2.70291842963}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Adrenal_Gland", "value": 0.0408306971307}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Adrenal_Gland", "value": 1.30477973502}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Adrenal_Gland", "value": -0.510189026083}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Adrenal_Gland", "value": -0.442475677509}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Adrenal_Gland", "value": -1.6326131209799999}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Adrenal_Gland", "value": -0.379181261369}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Adrenal_Gland", "value": -1.69176370211}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Adrenal_Gland", "value": -1.1486618814899998}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Adrenal_Gland", "value": -1.5825642688100001}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Adrenal_Gland", "value": -1.18835047506}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Adrenal_Gland", "value": -0.311441111109}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Adrenal_Gland", "value": 0.290836427203}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Adrenal_Gland", "value": -1.75119993814}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Adrenal_Gland", "value": 0.294647302225}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Adrenal_Gland", "value": 0.175715081794}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Adrenal_Gland", "value": -0.586720004633}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Adrenal_Gland", "value": -0.743871340308}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Adrenal_Gland", "value": -0.83380731514}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Pituitary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Pituitary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Pituitary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Pituitary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Pituitary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Pituitary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Pituitary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Pituitary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Pituitary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Pituitary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Pituitary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Pituitary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Pituitary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Pituitary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Pituitary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Pituitary", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Pituitary", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Pituitary", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Pituitary", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Pituitary", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Pituitary", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Pituitary", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Pituitary", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Pituitary", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Pituitary", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Pituitary", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Pituitary", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Pituitary", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Pituitary", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Pituitary", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Pituitary", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Pituitary", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Pancreas", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Pancreas", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Pancreas", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Pancreas", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Pancreas", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Pancreas", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Pancreas", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Pancreas", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Pancreas", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Pancreas", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Pancreas", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Pancreas", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Pancreas", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Pancreas", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Pancreas", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Pancreas", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Pancreas", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Pancreas", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Pancreas", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Pancreas", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Pancreas", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Pancreas", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Pancreas", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Pancreas", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Pancreas", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Pancreas", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Pancreas", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Pancreas", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Pancreas", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Pancreas", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Pancreas", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Pancreas", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Thyroid", "value": -4.673835131580001}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Thyroid", "value": 8.67093395598}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Thyroid", "value": 5.608348228790001}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Thyroid", "value": 4.0116173780199995}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Thyroid", "value": 5.96745259055}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Thyroid", "value": 3.33434988458}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Thyroid", "value": 2.7646186221099995}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Thyroid", "value": 5.121234402880001}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Thyroid", "value": 5.030649341919999}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Thyroid", "value": 4.94325926738}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Thyroid", "value": 1.15419624064}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Thyroid", "value": -10.0995662035}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Thyroid", "value": -9.917153638010001}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Thyroid", "value": 3.9339942409800006}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Thyroid", "value": -4.22241199888}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Thyroid", "value": 3.66975877644}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Thyroid", "value": -3.9905806906300003}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Thyroid", "value": -4.06482975333}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Thyroid", "value": -6.550390838769999}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Thyroid", "value": -6.304795065}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Thyroid", "value": -4.643972386230001}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Thyroid", "value": -4.69185097017}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Thyroid", "value": -4.49529345008}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Thyroid", "value": -4.56482179047}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Thyroid", "value": -8.98331725977}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Thyroid", "value": 5.98539852589}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Thyroid", "value": -1.2079767650700002}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Thyroid", "value": 4.430085957119999}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Thyroid", "value": 4.30355912424}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Thyroid", "value": -3.9133646422400004}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Thyroid", "value": -3.6669008034900004}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Thyroid", "value": -3.85015953985}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Lung", "value": -4.867917862780001}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Lung", "value": 8.80979215162}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Lung", "value": 5.220560239669999}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Lung", "value": 3.8206559015199995}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Lung", "value": 5.722374972359999}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Lung", "value": 3.7193921888199997}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Lung", "value": 3.32088557662}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Lung", "value": 5.2034956159400005}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Lung", "value": 4.61851464882}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Lung", "value": 4.42768047637}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Lung", "value": 0.880804963457}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Lung", "value": -10.4491540977}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Lung", "value": -10.0436939747}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Lung", "value": 3.5544899733699995}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Lung", "value": -3.9726091309199996}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Lung", "value": 3.21825819879}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Lung", "value": -3.4845227978500004}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Lung", "value": -3.6842367964599996}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Lung", "value": -7.612694563300001}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Lung", "value": -6.5261212413}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Lung", "value": -4.18749038291}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Lung", "value": -4.496622673269999}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Lung", "value": -4.08225035997}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Lung", "value": -4.37350313954}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Lung", "value": -9.05844351997}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Lung", "value": 4.906095773680001}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Lung", "value": -3.57497510694}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Lung", "value": 4.36383655806}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Lung", "value": 4.3765678773}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Lung", "value": -4.02096813165}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Lung", "value": -3.31473350602}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Lung", "value": -3.89303017414}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Ovary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Ovary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Ovary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Ovary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Ovary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Ovary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Ovary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Ovary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Ovary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Ovary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Ovary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Ovary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Ovary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Ovary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Ovary", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Ovary", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Ovary", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Ovary", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Ovary", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Ovary", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Ovary", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Ovary", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Ovary", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Ovary", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Ovary", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Ovary", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Ovary", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Ovary", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Ovary", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Ovary", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Ovary", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Ovary", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Prostate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Prostate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Prostate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Prostate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Prostate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Prostate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Prostate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Prostate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Prostate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Prostate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Prostate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Prostate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Prostate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Prostate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Prostate", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Prostate", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Prostate", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Prostate", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Prostate", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Prostate", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Prostate", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Prostate", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Prostate", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Prostate", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Prostate", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Prostate", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Prostate", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Prostate", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Prostate", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Prostate", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Prostate", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Prostate", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Testis", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Testis", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Testis", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Testis", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Testis", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Testis", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Testis", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Testis", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Testis", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Testis", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Testis", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Testis", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Testis", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Testis", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Testis", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Testis", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Testis", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Testis", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Testis", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Testis", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Testis", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Testis", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Testis", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Testis", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Testis", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Testis", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Testis", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Testis", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Testis", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Testis", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Testis", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Testis", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Uterus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Uterus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Uterus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Uterus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Uterus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Uterus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Uterus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Uterus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Uterus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Uterus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Uterus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Uterus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Uterus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Uterus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Uterus", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Uterus", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Uterus", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Uterus", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Uterus", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Uterus", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Uterus", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Uterus", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Uterus", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Uterus", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Uterus", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Uterus", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Uterus", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Uterus", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Uterus", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Uterus", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Uterus", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Uterus", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0594 (0.0166)", "phenotype": "Body fat percentage complete", "phenotype_link": "nopmid.html", "index": 0, "code": "BFPC04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5273, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 8.629691000000001, "LDSC Ratio": "0.0389 (0.0109)", "LDSC Lambda GC": 2.0217, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Body fat percentage complete (BFPC04) ", "LDSC Total Observed Scale h2": "0.2143 (0.0066)", "variable": "Vagina", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0021 (0.0255)", "phenotype": "red blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 1, "code": "BLOO02", "category": "g: blood", "LDSC Mean Chi-Square": 1.6473, "PMID or link": "27863252", "gene": "MAPK3", "association": 26.359171000000003, "LDSC Ratio": "0.0032 (0.0394)", "LDSC Lambda GC": 1.2464, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Red blood cell count (BLOO02) ", "LDSC Total Observed Scale h2": "0.1866 (0.0175)", "variable": "Vagina", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0155 (0.0407)", "phenotype": "reticulocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 2, "code": "BLOO04", "category": "g: blood", "LDSC Mean Chi-Square": 1.6377, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.615539, "LDSC Ratio": "0.0243 (0.0638)", "LDSC Lambda GC": 1.207, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Reticulocyte count (BLOO04) ", "LDSC Total Observed Scale h2": "0.1686 (0.0194)", "variable": "Vagina", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0465 (0.0338)", "phenotype": "monocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 3, "code": "BLOO05", "category": "g: blood", "LDSC Mean Chi-Square": 1.6776, "PMID or link": "27863252", "gene": "MAPK3", "association": 9.509171, "LDSC Ratio": "0.0686 (0.0499)", "LDSC Lambda GC": 1.2201, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Monocyte count (BLOO05) ", "LDSC Total Observed Scale h2": "0.1863 (0.0217)", "variable": "Vagina", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9587 (0.0273)", "phenotype": "eosinophil count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 4, "code": "BLOO07", "category": "g: blood", "LDSC Mean Chi-Square": 1.6196, "PMID or link": "27863252", "gene": "MAPK3", "association": 16.578823999999997, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil count (BLOO07) ", "LDSC Total Observed Scale h2": "0.1808 (0.0221)", "variable": "Vagina", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9683 (0.0197)", "phenotype": "lymphocyte count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 5, "code": "BLOO11", "category": "g: blood", "LDSC Mean Chi-Square": 1.6107, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.748922, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte count (BLOO11) ", "LDSC Total Observed Scale h2": "0.1825 (0.0141)", "variable": "Vagina", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9648 (0.0192)", "phenotype": "white blood cell count", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 6, "code": "BLOO12", "category": "g: blood", "LDSC Mean Chi-Square": 1.5935, "PMID or link": "27863252", "gene": "MAPK3", "association": 8.634924, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2431, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "White blood cell count (BLOO12) ", "LDSC Total Observed Scale h2": "0.1772 (0.0165)", "variable": "Vagina", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9694 (0.0258)", "phenotype": "Sum eosinophil basophil counts", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 7, "code": "BLOO16", "category": "g: blood", "LDSC Mean Chi-Square": 1.5912, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.947151999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1908, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Sum eosinophil basophil counts (BLOO16) ", "LDSC Total Observed Scale h2": "0.1712 (0.0201)", "variable": "Vagina", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9613 (0.0261)", "phenotype": "Eosinophil percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 8, "code": "BLOO17", "category": "g: blood", "LDSC Mean Chi-Square": 1.5997, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.717808999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of white cells (BLOO17) ", "LDSC Total Observed Scale h2": "0.1752 (0.02)", "variable": "Vagina", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9588 (0.0243)", "phenotype": "Eosinophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 9, "code": "BLOO18", "category": "g: blood", "LDSC Mean Chi-Square": 1.5779, "PMID or link": "27863252", "gene": "MAPK3", "association": 13.344372, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1876, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Eosinophil percentage of granulocytes (BLOO18) ", "LDSC Total Observed Scale h2": "0.1696 (0.019)", "variable": "Vagina", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9664 (0.0153)", "phenotype": "Lymphocyte percentage of white cells", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 10, "code": "BLOO24", "category": "g: blood", "LDSC Mean Chi-Square": 1.4799, "PMID or link": "27863252", "gene": "MAPK3", "association": 1.576984, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.2038, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Lymphocyte percentage of white cells (BLOO24) ", "LDSC Total Observed Scale h2": "0.1474 (0.0115)", "variable": "Vagina", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0092 (0.0408)", "phenotype": "Mean corpuscular hemoglobin MCH", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 11, "code": "BLOO26", "category": "g: blood", "LDSC Mean Chi-Square": 1.7671, "PMID or link": "27863252", "gene": "MAPK3", "association": 31.679729, "LDSC Ratio": "0.012 (0.0532)", "LDSC Lambda GC": 1.194, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular hemoglobin MCH (BLOO26) ", "LDSC Total Observed Scale h2": "0.2208 (0.0255)", "variable": "Vagina", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0078 (0.0383)", "phenotype": "Mean corpuscular volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 12, "code": "BLOO27", "category": "g: blood", "LDSC Mean Chi-Square": 1.8174, "PMID or link": "27863252", "gene": "MAPK3", "association": 29.107065000000002, "LDSC Ratio": "0.0096 (0.0469)", "LDSC Lambda GC": 1.2136, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean corpuscular volume (BLOO27) ", "LDSC Total Observed Scale h2": "0.2364 (0.0256)", "variable": "Vagina", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0051 (0.0458)", "phenotype": "Mean platelet volume", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 13, "code": "BLOO29", "category": "g: blood", "LDSC Mean Chi-Square": 1.901, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.640544, "LDSC Ratio": "0.0057 (0.0508)", "LDSC Lambda GC": 1.2005, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Mean platelet volume (BLOO29) ", "LDSC Total Observed Scale h2": "0.2652 (0.0382)", "variable": "Vagina", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "0.9571 (0.0217)", "phenotype": "Neutrophil percentage of granulocytes", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 14, "code": "BLOO31", "category": "g: blood", "LDSC Mean Chi-Square": 1.5319, "PMID or link": "27863252", "gene": "MAPK3", "association": 11.473299, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1843, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Neutrophil percentage of granulocytes (BLOO31) ", "LDSC Total Observed Scale h2": "0.1586 (0.0172)", "variable": "Vagina", "value": 0.0}, {"study size": 173480, "LDSC Intercept": "1.0176 (0.0246)", "phenotype": "Plateletcrit", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/27863252", "index": 15, "code": "BLOO33", "category": "g: blood", "LDSC Mean Chi-Square": 1.6946, "PMID or link": "27863252", "gene": "MAPK3", "association": 5.5066120000000005, "LDSC Ratio": "0.0253 (0.0354)", "LDSC Lambda GC": 1.2498, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Plateletcrit (BLOO33) ", "LDSC Total Observed Scale h2": "0.2019 (0.0171)", "variable": "Vagina", "value": 0.0}, {"study size": 339224, "LDSC Intercept": "0.6766 (0.01)", "phenotype": "BMI (Mixed ancestry Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 16, "code": "BODY03", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.321, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.971794999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.1301, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (Mixed ancestry Locke) (BODY03) ", "LDSC Total Observed Scale h2": "0.1433 (0.0079)", "variable": "Vagina", "value": 0.0}, {"study size": 322154, "LDSC Intercept": "0.6475 (0.0096)", "phenotype": "BMI (European Locke)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673413", "index": 17, "code": "BODY04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.2586, "PMID or link": "25673413", "gene": "MAPK3", "association": 5.510056, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0741, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (European Locke) (BODY04) ", "LDSC Total Observed Scale h2": "0.1374 (0.0077)", "variable": "Vagina", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.0353 (0.0162)", "phenotype": "BMI", "phenotype_link": "nopmid.html", "index": 18, "code": "BODY08", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.6286, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 22.787253, "LDSC Ratio": "0.0217 (0.01)", "LDSC Lambda GC": 2.0471, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "BMI (BODY08) ", "LDSC Total Observed Scale h2": "0.23 (0.0081)", "variable": "Vagina", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.043 (0.0163)", "phenotype": "Fat mass", "phenotype_link": "nopmid.html", "index": 19, "code": "FATM04", "category": "b: anthropometric", "LDSC Mean Chi-Square": 2.5512, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 16.593238, "LDSC Ratio": "0.0277 (0.0105)", "LDSC Lambda GC": 2.0175, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat mass (FATM04) ", "LDSC Total Observed Scale h2": "0.2218 (0.0073)", "variable": "Vagina", "value": 0.0}, {"study size": 88019, "LDSC Intercept": "0.817 (0.0077)", "phenotype": "Hip circumference European female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 20, "code": "HIPC01F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.105, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.560936, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European female (HIPC01F) ", "LDSC Total Observed Scale h2": "0.1742 (0.0104)", "variable": "Vagina", "value": 0.0}, {"study size": 145308, "LDSC Intercept": "0.7568 (0.0085)", "phenotype": "Hip circumference European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 21, "code": "HIPC01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1677, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.435796000000001, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference European (HIPC01) ", "LDSC Total Observed Scale h2": "0.1493 (0.0086)", "variable": "Vagina", "value": 0.0}, {"study size": 88035, "LDSC Intercept": "0.8148 (0.0077)", "phenotype": "Hip circumference All Ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 22, "code": "HIPC02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1055, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.208561999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries female (HIPC02F) ", "LDSC Total Observed Scale h2": "0.1759 (0.0106)", "variable": "Vagina", "value": 0.0}, {"study size": 145414, "LDSC Intercept": "0.7536 (0.0085)", "phenotype": "Hip circumference All Ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 23, "code": "HIPC02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1656, "PMID or link": "25673412", "gene": "MAPK3", "association": 8.572774, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Hip circumference All Ancestries (HIPC02) ", "LDSC Total Observed Scale h2": "0.1491 (0.0085)", "variable": "Vagina", "value": 0.0}, {"study size": 353972, "LDSC Intercept": "1.11 (0.0274)", "phenotype": "Fat free mass / lean mass", "phenotype_link": "nopmid.html", "index": 24, "code": "LEAN06", "category": "b: anthropometric", "LDSC Mean Chi-Square": 3.1445, "PMID or link": "Not published (C. H\u00fcbel 2018 UK Biobank)", "gene": "MAPK3", "association": 40.661524, "LDSC Ratio": "0.0513 (0.0128)", "LDSC Lambda GC": 2.1851, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Fat free mass / lean mass (LEAN06) ", "LDSC Total Observed Scale h2": "0.2904 (0.0116)", "variable": "Vagina", "value": 0.0}, {"study size": 329345, "LDSC Intercept": "1.0084 (0.0147)", "phenotype": "Age at Menarche (marker of puberty in females)", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/28436984", "index": 25, "code": "MENA02F", "category": "j: other", "LDSC Mean Chi-Square": 2.0433, "PMID or link": "28436984", "gene": "MAPK3", "association": 11.08008, "LDSC Ratio": "0.0081 (0.0141)", "LDSC Lambda GC": 1.6108, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Age at Menarche (marker of puberty in females) (MENA02F) ", "LDSC Total Observed Scale h2": "0.1603 (0.0081)", "variable": "Vagina", "value": 0.0}, {"study size": 79799, "LDSC Intercept": "0.9435 (0.0122)", "phenotype": "Rheumatoid Arthritis 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/24390342", "index": 26, "code": "RHEU01", "category": "h: musculoskeletal", "LDSC Mean Chi-Square": 1.1179999999999999, "PMID or link": "24390342", "gene": "MAPK3", "association": 6.460459999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Rheumatoid Arthritis 2014 (RHEU01) ", "LDSC Total Observed Scale h2": "0.1087 (0.0197)", "variable": "Vagina", "value": 0.0}, {"study size": 82315, "LDSC Intercept": "1.033 (0.0104)", "phenotype": "Schizophrenia PGC 2014", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 27, "code": "SCHI01", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.755, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.178978, "LDSC Ratio": "0.0437 (0.0138)", "LDSC Lambda GC": 1.5696, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 (SCHI01) ", "LDSC Total Observed Scale h2": "0.4467 (0.0157)", "variable": "Vagina", "value": 0.0}, {"study size": 77096, "LDSC Intercept": "1.0382 (0.0103)", "phenotype": "Schizophrenia PGC 2014 European", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25056061", "index": 28, "code": "SCHI02", "category": "c: mental and behavioural", "LDSC Mean Chi-Square": 1.7409999999999999, "PMID or link": "25056061", "gene": "MAPK3", "association": 6.2500160000000005, "LDSC Ratio": "0.0515 (0.0139)", "LDSC Lambda GC": 1.5622, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Schizophrenia PGC 2014 European (SCHI02) ", "LDSC Total Observed Scale h2": "0.465 (0.0168)", "variable": "Vagina", "value": 0.0}, {"study size": 231927, "LDSC Intercept": "0.7423 (0.0086)", "phenotype": "Waist circumference EU", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 29, "code": "WAIS01", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1487, "PMID or link": "25673412", "gene": "MAPK3", "association": 6.277325, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference EU (WAIS01) ", "LDSC Total Observed Scale h2": "0.1411 (0.0084)", "variable": "Vagina", "value": 0.0}, {"study size": 113882, "LDSC Intercept": "0.8087 (0.0072)", "phenotype": "Waist circumference all ancestries female", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 30, "code": "WAIS02F", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.0852, "PMID or link": "25673412", "gene": "MAPK3", "association": 5.881405, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries female (WAIS02F) ", "LDSC Total Observed Scale h2": "0.1618 (0.01)", "variable": "Vagina", "value": 0.0}, {"study size": 244261, "LDSC Intercept": "0.7406 (0.0087)", "phenotype": "Waist circumference all ancestries", "phenotype_link": "https://www.ncbi.nlm.nih.gov/pubmed/25673412", "index": 31, "code": "WAIS02", "category": "b: anthropometric", "LDSC Mean Chi-Square": 1.1516, "PMID or link": "25673412", "gene": "MAPK3", "association": 7.295000999999999, "LDSC Ratio": "Ratio < 0 (usually indicates GC correction).", "LDSC Lambda GC": 1.0466, "ENSEMBL": "ENSG00000102882", "phenotype_reference": "Waist circumference all ancestries (WAIS02) ", "LDSC Total Observed Scale h2": "0.1425 (0.0084)", "variable": "Vagina", "value": 0.0}]}};
    var embed_opt = {"mode": "vega-lite"};

    function showError(el, error){
        el.innerHTML = ('<div class="error">'
                        + '<p>JavaScript Error: ' + error.message + '</p>'
                        + "<p>This usually means there's a typo in your chart specification. "
                        + "See the javascript console for the full traceback.</p>"
                        + '</div>');
        throw error;
    }
    const el = document.getElementById('vis');
    vegaEmbed("#vis", spec, embed_opt)
      .catch(error => showError(el, error));
  </script>
</body>
</html>
